
# Covid19

* [Public Description](#public-description)
* [Infections and Diseases](#infections-and-diseases)
* [Disease Factors](#disease-factors)
* [Infection Locations](#infection-locations)
  * [Common Infection Location](#common-infection-location)
* [Immune Response and Symptoms](#immune-response-and-symptoms)
  * [Causes of Symptoms](#causes-of-symptoms)
  * [Localized Symptoms](#localized-symptoms)
* [Disease Stages](#disease-stages)
* [Diagnosis](#diagnosis)
  * [History of Covid-19 Diagnosis](#history-of-covid-19-diagnosis)
  * [Methods to Directly Detect Viruses](#methods-to-directly-detect-viruses)
  * [Methods to Indirectly Detect Viruses](#methods-to-indirectly-detect-viruses)
  * [Diagnosis Locations](#diagnosis-locations)
  * [Common Diagnosis Methods](#common-diagnosis-methods)
  * [Challenges regarding Viral Load and Diagnosis](#challenges-regarding-viral-load-and-diagnosis)
* [Viral Load](#viral-load)
  * [Local Amplification](#local-amplification)
  * [Relevance of Viral Load](#relevance-of-viral-load)
  * [Types of Specimens](#types-of-specimens)
  * [Viral Load at different Locations](#viral-load-at-different-locations)
  * [Tissues](#tissues)
* [Salvia based Diagnosis](#salvia-based-diagnosis)
  * [Literature on Salvia Diagnosis](#literature-on-salvia-diagnosis)
  * [Viral Load in Salvia](#viral-load-in-salvia)
* [Fatality Rate](#fatality-rate)
  * [Estimates](#estimates)
* [Summarized References](#summarized-references)
  * [Diagnosis](#diagnosis-1)
  * [References RNA Detection Stability](#references-rna-detection-stability)
  * [Time Series](#time-series)
* [References](#references)
* [Appendix](#appendix)


## Public Description
Covid-19 has different clinical pictures and [symptoms](#immune-response-and-symptoms). The appearances vary since they depend on the [locations](#infection-locations) infected andq SARS-CoV-2 can infect many locations in the respiratory tract except the vocal folds. Therefore any kind of respiratory symptoms - except a hoarse voice - can indicate Covid-19. With such symptoms one should avoid crowded indoor spaces and do a Covid test. 
Many of the symptoms are not specific to Covid-19, they may have other causes than SARS-CoV-2: having a (respiratory) infection with different virus or a bacterium, having eaten/drunken something harmful (e.g. rotten or too much alcohol and the like)or just not having had enough sleep.



## Infections and Diseases

> ##### Disease
> A *disease* is a biological disorder of structure or function in an organism causing inconvenience and/or unwanted effects. Literally disease means without-ease (dis=without, separated from). A disease is often dynamic and an immune response is induced/involved.

Chemical/biochemical damages such as dehydration, a poisoning or physical damage such as a fracture or an injury are not diseases themselves but chemical/physical damage can result from a disease.

There are many different causes of diseases:
* other [life forms](./coronavirus.md#life-form) interfering with the metabolism, such an interference is called [infection as defined below](#virus-infection). 
  * outside of cells but inside multicellular organisms
    * most bacteria
    * fungi
  * inside cells
    * some bacteria
    * all viruses
* immune disease in which the immune system overreacts to substances without need (notes on the immune system in the [Immune System chapter](./immune_system.md))

Virus diseases are cause by virus infections:

> ##### Virus Infection
> A *virus infects* an organism by infecting cells of the organism. A virus infects a cell by entering it and changing/hijacking the cells metabolism to produce virus progeny - called virions. More about viruses and their infections in the chapter [Introduction to Viruses](./coronavirus.md#introduction-to-viruses)

If a virus infection causes lack of ease, one has a virus disease:

> ##### Virus Disease
> A *virus disease* is a [virus infection of an organism](./coronavirus.md#virus-infection) causing adverse effects. 

Adverse effects result either directly through tissue damage caused by viruses or more frequent to keep the body virus free (to prevent future cell/tissue damage) a strong immune response is induced causing effects on its own.

The definition of a virus disease is not strict:  
* Adverse effects or tissue damage are often small and no lack of ease is caused. For most viruses very mild infections are not considered diseases. 
* For viruses which can be highly pathogenic usually one speaks of a disease if there is an infection diagnosed regardless whether there are symptoms or adverse effects.

> ##### Covid-19
> *Covid-19* stands for COronaVIrus Disease 2019. As of December 2020, all SARS-CoV-2 infections are called Covid-19. 

In colloquial language, "Covid-19" is often used synonymously with having SARS-CoV-2 viruses detected in the upper respiratory tract, often tested by nasopharyngeal sampling and PCR testing. However the situation is not that simple:  
* Not all persons infected with SARS-CoV-2 have a detectable amount of viruses in the nose since SARS-CoV-2 can infect the lower respiratory tract only. Especially people with a sensible immune system may experience symptoms without having a detectable viral load in the upper respiratory tract. But it is also possible to have severe Covid-19 without having detectable viruses by nose swabs [citation in work].  
* On the contrary some people have a detectable viral load in the upper respiratory tract but no symptoms. The following terms describe the symptom stage of a person.

More about in the sections [Diagnosis](#diagnosis) and Locations](#infection-locations).

> ##### Symptomatic
> A person experiencing/notifying/showing symptoms of a disease is called symptomatic. 
> ##### Asymptomatic
> A person *not* experiencing/notifying/showing symptoms while having an infection is called asymptomatic.
> ##### Presymptomatic
> Even those person developing symptoms are asymptomatic in the early phase of a disease during these phase a person is called presymptomatic. 

## Disease Factors
A virus disease is an interplay of host factors including the host immune system, the behavior of the virus and of environmental factors.

* __Host Factors__ relevant for the tropism of Covid-19 are
    - Cells susceptible to infection and capable of producing viruses (not all cells are susceptible e.g. red blood cells are not).
    - Host [immune response](#immune-response-and-symptoms).
* __Capabilities and Behavior of the Virus__
  Viruses behave different under different environmental factors (called tropism of viruses), described in the section [Tropism of viruses](./coronavirus.md/tropism). E.g. SARS-2 virions can initiate cell entry only for cells containing a matching receptor. To infect a tissue several conditions must be met as described in the chapter [Tissue Tropism of SARS-CoV-2](./coronavirus.md/#tissue-tropism-of-sars-cov-2).
* __Environmental Factors__
    - the travel of virions between hosts and thus the mode of the initial infection. E.g. in dry air exhaled droplets dry out and these are more likely to be deposited deep in the lungs.
    - Environmental factors can influence the host factors as well as the virus tropism and thus indirectly the course of the disease. E.g.:
      - Very dry air can weaken the immune system of the mucous membrane in the respiratory tract.
      - Dirty air can weaken defence mechanisms against viruses e.g. ciliated cells are damaged and thereby the clearance of particles in the respiratory tract is reduced.



## Infection Locations
SARS-CoV-2 infects different tissues in humans. The effects from the virus infection such as tissue damage and symptoms are often local to the infected locations.

Different topics related to the infection locations are described in the sections:
* The section [Viral Load](#viral-load) describes in which body fluids and tissues how many virions are observed.
* The [Immune Response and Symptoms](#immune-response-and-symptoms) section gives an overview of the different symptoms induced by the infection location.
* On the [coronavirus page](./coronavirus.md), the section [tissue tropism](./coronavirus.md#tissue-tropism-of-sars-cov-2) describes the tissue preferences of coronaviruses and SARS-CoV-2 especially.
* The experimental page [Coronavirus Disease Pattering](./ideas_to_explore/coronavirus_disease_patterns.md) describes that coronavirus infections at different locations can be experienced/observed as different diseases.

### Common Infection Location
* Upper Respiratory Tract => Symptoms are similar to other respiratory diseases: a sore throat, coughing and a runny nose. Loss of smell or taste.
* Throat => Coughing, the body wants to get rid of the viruses in the respiratory tract.
* Central Bronchi => Dry cough.
* Deep Lungs i.e. small bronchi, alveoli => If the lungs get involved, symptoms can be a strange feeling in the chest like something sticky being inside (dry mucus loaded with viruses) or even shortness of breath (the lungs are inflamed). Diarrhea can both start early and later on and can outlast the respiratory infection by days or even weeks.
* virions in the alveoli and spread to the blood => Early on virions are released into the blood. This feels like it is starting from within. Symptoms can be chills (because the body is increasing the temperature to fight the viruses) or just strange feeling, similar to a weak poisoning when having eaten something rotten or drunken too much. Feeling tired, energy-less and having an increased recovery time after sports are indications too.



## Immune Response and Symptoms
[in work]

### Causes of Symptoms
The symptoms of a [virus disease](#virus-disease) are caused:
1. Direct symptoms through cell damage (rare)
2. Indirect symptoms through the induced [immune response](./immune_system.md) (frequent).

### Localized Symptoms
Coronavirus infections can be localized (especially in the early phase), therefore the symptoms can be local and depend on the locations infected as described in the section [Infection Locations and Symptoms](#infection-location-and-symptoms). 


## Disease Stages
[in work]


## Diagnosis

### History of Covid-19 Diagnosis
* In late 2019 and early 2020 Covid-19 was often diagnosed based on lung CT findings.
* From February 2020 to December 2020 the most used method to diagnose Covid-19, is having a detectable amount of SARS-CoV-2 in nasopharyngeal swabs. The viral load is detected either by [PCR-Test](#pcr-test) or [Antigen-Tests](#antigen-test). In the early phase of the Covid-19 pandemics when the bottleneck was the availability of accurate PCR tests. 
* Late 2020 antigen tests are getting more common.
* 2021? Alternate Locations? [Rethink Diagnosis](#diagnosis-locations) and a chapter on its own devoted to [salvia based diagnosis](#salvia-based-diagnosis).


### Methods to Directly Detect Viruses
In tissues containing infected cells or in the lining fluids protecting/cleaning these tissues the viral load can be measured.
* #### Detect Building Blocks of Viruses
  To detect viruses replicating the building blocks of virions can be searched.
  * ##### PCR Test 
    Test for specific sequences in the genome of virions (positive single stranded RNA for coronaviruses). The amount of virions - called [viral load](#viral-load) - can be determined with high specificity and sensitivity. Examples are given in the section [Common Diagnosis Methods](#common-diagnosis-methods).
  * ##### Antigen Test
    Methods detecting proteins of virions are usually called antigen tests (virus proteins are called antigens since these proteins can be recognized by the immune system as harmful).
    * There are many different products of antigen tests which vary a lot in sensitivity and specificity.
    * The sensitivity and specificity of antigen tests tend to be lower than PCR based tests for most antigen test kits available as of December 2020.
* #### Detect Metabolism of Coronaviruses
  Temporary RNA produced from coronaviruses with a running metabolism can be detected by [PCR-Tests](#pcr-test).
* #### Detect Infectious Virions
  [in work: to be checked and incomplete]
   Samples possibly containing virions can be put on cell or organ cultures. These cell or organ cultures then can be inspected by another methods such as detecting tissue change/damage or detecting RNA to see whether there were infectious virions in the  samples. The samples can be diluted so much that virus colonies originating from single virions can be distinguished. In this  way the number of infectious virions can be determined (this method is called Plaque assay)


### Methods to Indirectly Detect Viruses
* #### Detect Tissue damage
  * chemical e.g. detecting cell damage by detecting distinctive proteins.
    (-) not specific
  * optical e.g. round glassy occupancies in the lung are characteristic for Covid-19 [in work and to check/specify and cite]
    (+) quite specific in the lung
    (-) in the upper respiratory tract, tissue damage to low
* #### Detect Immune Response
  * #### Antibody Test
    [in work] <!--td: update immune sys page with antibodies>
    For infections triggering certain alarms, the immune response involves antibodies which can be detected in the blood. The antibodies are in the serum of the blood. Methods analyzing the blood serum are called *serology* and often antibody test are referred serological (Covid) tests. There are different kind of antibodies which are produced at different stages during and after an infection. Infections in later stages and past infections can be diagnosed by this method. 


### Diagnosis Locations
Two types of locations are suitable for diagnosis: 
* Tissues with active viral replication ([Infected Locations](#infection-locations))
* Viral Load in body fluids ([Viral Load](#viral-load)) from virus replication in close tissues or deposition. 

The fluids in the respiratory tract are a good locations to detect SARS-CoV-2 virions:
* Since SARS-CoV-2 infects and transmits mainly through the respiratory tract, the respiratory tracts is the place to search
* The virions mostly are released on the apical side and therefore there are virions in the fluids covering infected tissues. 
* For accessibility reasons fluids in the upper respiratory are preferred. However the largest number of susceptible cells to SARS-CoV-2 are in the lower respiratory tract.

### Common Diagnosis Methods
* [Nasopharyngeal Swabs](#nasopharyngeal-swabs) 
* [Salvia Diagnosis](#salvia-diagnosis)
* [Lung CT](#lung-ct)

#### Nasopharyngeal Swabs
Nasopharyngeal Swabs (NPS) are, as of december 2020, often used as 'gold standard' for Covid-19 diagnosis. NPS samples can be analyzed either by PCR or antigen tests.
(+/-) quite easy to take but trained persons are needed
(+) the sensitivity is good for severe infections.

#### Salvia
Described in the section [Salvia based Diagnosis](#salvia-based-diagnosis).

#### Lung CT
Round glassy occupancies in the lung are characteristic for Covid [to check and cite]
(+) specific
(+) detects infection of the lung regardless whether there are viruses in the upper respiratory tract
(-) radiation exposure for patient
(-) only in diseases involving the lung significantly and thus sensitivity low
=> recommended for symptomatic risk patients of very high age (radiation exposure less relevant) with negative test in the upper respiratory tract


### Challenges regarding Viral Load and Diagnosis
Often investigations for viral load are biased. Reasons are:
* Sampling is biased towards symptomatic patients. Not necessarily bad since symptomatic patients are more infectious ([Section Infectors and Symptoms](./spread_analyses.md#infectors-and-symptoms) in the Chapter Spread Analyses) and more at risk for severe disease.
* Often NPS samples are taking as a 'Gold Standard' to diagnose Covid-19. Measuring viral load e.g. in salvia is often compared to NPS viral load and sometimes NPS sampling is assigned a sensitivity of 100%. 
  > I guess/estimate the sensitivity of NPS testing at a single time point lower than 75%
* E.g. not to every infection in the upper respiratory tract the body responds with antibodies or only antibodies of a specific type. On the other hand a lower respiratory tract infection may yield antibodies, but be negative at a single PCR in the upper respiratory tract.



## Viral Load
[**in work and unfinished**]

> ##### Viral Load
> The *viral load* denotes how many virions are present in specimen of tissue or body fluid.

A viral load is observed in locations with active viral replication. Sometimes a viral load can be detected upon deposition of virions from other infected locations. While the latter is necessary for the initial infection event of a location, the viral load through deposition is often to low to be detected. However often happens local amplification. 


### Local Amplification
While the lung is primarily infected and both responsible for the spread to other persons and the symptoms, there are susceptible cells in the nose and in the salviary glands for SARS-CoV-2 replication. Deposited virions can replicate and so the viral load is amplified in these locations. Therefore the nose mucus or salvia often have a detectable viral load and thus are suitable for diagnosis. 

### Relevance of Viral Load
Knowing the viral load observed in different fluids and tissues is relevant for:
* __Diagnosis__: For an optimal diagnosis location, the viral load is optimally high for the different disease patterns caused by SARS-CoV-2 and taking specimens is easy.  
* __Transmission__: Knowing the viral load in different locations (fluids:relevant for spread and infection and tissues:where the viruses replicate) can help to understand and prevent transmissions of SARS-CoV-2 virions.
* __Understanding Covid-19__: Knowing at which stages which locations are infected by the coronaviruses can help to predict the disease progression and to optimize the treatment. 



### Types of Specimens

#### Body Fluids
Coronaviruses virions are mostly released on the apical surfaces and so the virions in the fluids covering infected cells.

#### Tissues
Since sampling fluids is enough in most cases tissues are rarely collected. Sometimes tissues are analyzed post mortem which show the virus distribution in deadly SARS-CoV-2 infections. 



### Viral Load at different Locations

[in work] A recommended review for the viral load in upper respiratory tract specimens is by [Lee et al](#summary-review-lee).
#### Viral Load in the Mouth 
see [Viral Load in Salvia](#viral-load-in-salvia) in the section [Salvia based Diagnosis](#salvia-based-diagnosis)

#### Posterior Nose Mucus
Is obtained by nasopharyngeal swabs and the most used location for detecting Covid-19 (as of December 2020).
#### Anterior Nose Mucus
[in work]
In the review of  [Lee et al](#summarized-review-lee) some studies investigating the anterior nose mucus are cited.
#### Sputum
[in work]

#### Broncho-alveolar-lavage fluid (BAL)
One of the few locations in the lower respiratory tract which is/can be sampled. Despite the fact that the lower respiratory tract is the most relevant location for viral replication, the viral load is less known especially for mild cases since sampling is complicated. 

#### Blood

### Tissues
[on todo list]
#### Lung
#### Kidney
#### Intestine
#### Brain


## Salvia based Diagnosis
The salvia based diagnosis has advantages ([Matic et al](#summary-matic) describe this concisely in their introduction):
* Salvia since easier to sample than the back of the nose and suitable for self collection. 
* The detection rate of SARS-CoV-2 in salvia mixed with cough up sputum seems to perform as well or even better as NPS ([summary Lee](#summarized-review-lee))

but also challenges:
* The viral load in salvia depends on the exact method and location of sampling (e.g. throat wash, coughing up sputum) and the time taken (e.g. the salvia flow is increased during eating and decreased at night).

### Literature on Salvia Diagnosis
Summaries of the papers are in the section [Summarized References](#summarized-references).
* Reviews are 
  * [Lee et al](#summarized-review-lee)(has nice plots comparing the different methods) 
  * [da Silva](#summarized-review-dasilva)(helpful summaries of studies).
* A interesting read is [Yokota 2](#summary-yokota-2). In his [Opinion Paper](#summary-opinion-by-azzi) L. Azzi opts for salvia based diagnosis and summarizes key points of [Yokota et al](#summary-yokota-2).



### Viral Load in Salvia
The viral load in salvia depends on the exact method and location of sampling (e.g. throat wash, coughing up sputum) and the time taken (e.g. the salvia flow is increased during & after eating and decreased at night). 

#### Viral Load for different Sampling Procedures
[in work]
Different sampling procedures from different studies are listed and compared in the review of [Lee et al](#summarized-review-lee). 
* ##### Throat Salvia
  * Testees in [Huber et al](#summary-huber) were asked to clear the throat. A first group of testees were asked to clear the throat thoroughly (“Basic”, N = 835) and a second group to clear it three times (“Enhanced”, N = 352). A tendency for the enhanced protocol to yield a slightly higher sensitivity is observed.
* ##### Coughed up Sputum
* ##### Early Morning Salvia
* ##### Saline Gargle
  [Berenger et al](#summary-berenger) hast testees in a first round gargle saline water and in a second round to accumulate and then spit salvia and add universal transport medium (UTM). The sensitivity observed in the UTM method is higher than in the saline gargle method.
#### Viral Load compared to NPS
The viral load in salvia is similar to the viral load in the nose with an  agreement of about 90%. Since the viral load in salvia depends on the sampling methods, the results obtained in different studies ([Literature](#literature-on-salvia-diagnosis)) vary slightly.



## Fatality Rate
[in work]
The death rate depends on many factors and varies considerably.

The death rate calculation depends on how a Covid-19 infection is [defined](#infections-and-diseases) and [diagnosed](#diagnosis).

> ###### Infection Fatality Rate
> The *infection fatality rate* denotes the proportion how many people die from the infection from all people having the infection. 
> `ifr = #(deaths)/#(all infected)`

> *Fatality rate* on this page is used synonymously to [infection fatality rate](#infection-fatality-rate).

To define and diagnose an infection is not simple as described in [Diseases and Infections](#infections-and-diseases). Acute [Covid-19](#covid-19) is often diagnosed either by PCR test and past Covid-19 by seroprevalence of antibodies. These methods don't agree in all cases: One can have a positive PCR test but no antibodies formation or having antibodies but a negative PCR test in the nose (especially if measured only at a single time point).


> ###### Case Fatality Rate
> The *case fatality rate* denotes the death rate obtained by using the official case counts: `cfr = #deaths/#(official cases)`
 

The official Covid-19 case counts are in most countries fare to low and thus the cfr fare to high. 
> The mismatch between the cfr can ifr can be exploited to estimate the `detection rate = #(case count)/#(actual infected)` and the hidden cases:  
`cfr/ifr = [#(deaths)/#(all infected)] / [#(deaths)/#(cases count)] = #(case count)/#(all infected) = detection rate`  
(more in the section [Detect Estimates Out of Bounds](./monitoring.md#detect-estimates-out-of-bounds) in the [Monitoring chapter](./monitoring.md)).

### Estimates
[in work]
* John Ioannidis infers the death rate from seroprevalence data across the world. Results:
  * 0.27% is the median COVID-19 infection fatality rate   (0.23% if corrected for the number of antibody types tested (immunoglobin, IgG, IgM, IgA))
  * 0.09% infection-fatality-rate in locations with COVID-19 population-mortality-rates less than the global average (< 118 deaths/million)
  * 0.20% infection-fatality-rate in locations with 118–500 COVID-19 deaths/million people
  * 0.57% infection-fatality-rate in locations with > 500 COVID-19 deaths/million people
  * 0.05% infection-fatality-rate corrected median for people younger than 70 years



## Summarized References
As usually for summarized references:
* in "..." are word-for-word extracts. 
* "Figure 1" and similar refer to figures in the corresponding paper. 
* While trying to capture the key findings of the investigations, the summaries are biased towards the findings referred on this page. 

### Diagnosis

#### Summarized Review Lee
Lee, Rose & Herigon, Joshua & Benedetti, Andrea & Pollock, Nira & Denkinger, Claudia. (2020). **Performance of Saliva, Oropharyngeal Swabs, and Nasal Swabs for SARS-CoV-2 Molecular Detection: A Systematic Review and Meta-analysis.** 10.1101/2020.11.12.20230748. 
A review and meta analysis. Helpful plots showing uniformly the detection rates of SARS-CoV-2 in NPS/salvia/NS for the reviewed studies and aggregated detection rates for different features (e.g. symptomatic) and different collection methods (NPS/Salvia/NS). 
##### Methods
"We systematically searched PubMed, Google Scholar, medRxiv, and bioRxiv (last retrieval October 1st, 2020) for comparative studies of alternative specimen types [saliva, oropharyngeal (OP), and nasal (NS) swabs] versus NP swabs for SARS-CoV-2 diagnosis using nucleic acid amplification testing (NAAT)." => "From 1,253 unique citations, we identified 25 saliva, 11 NS, 6 OP, and 4 OP/NS studies meeting 15 inclusion criteria."
##### Findings
The results from the different studies/data aggregated by feature are summarized to a figure which shows % positive alternate specimens, % positive NP specimens and (if available) % specimens where both NP and alternate are positive. 
Selected aggregated results, the complete list is found in Figure 2 in the paper

> Note: Sensitivity values should be taken as a score and *not* as detection rates in percentages since diagnosis by upper respiratory specimens alone is not possible.

* Overall salvia they conclude a detection rate of 88% for salvia and 94% for NPS //both detection rates are likely much lower see Limitations
* Cough or deep  throat salvia have a detection rate of 94% while NPS have 89%.
* Asymptomatic patients have a detection rate of 87% in salvia and 73% in NPS
* Symptomatic Patients have a detection rate of 88% in salvia and 96% in NPS



#### Summarized Review daSilva 
Medeiros da Silva, R. C., Nogueira Marinho, L. C., de Araújo Silva, D. N., Costa de Lima, K., Pirih, F. Q., & Luz de Aquino Martins, A. R. (2020). **Saliva as a possible tool for the SARS-CoV-2 detection: A review.** Travel medicine and infectious disease, 38, 101920. 

Helpful summaries of 39 studies, which analyze the viral load in salvia. 
##### Methods
Search for (“saliva”) and (“SARS-CoV-2” or “coronavirus” or “COVID-1”) in PubMed, Medline, Cochrane Library, Web of Science, Embase and Scopus yielded:
"A total of 363 studies were identified by  and 39 were selected for review."
##### Findings
* For most studies: Saliva samples yielded a viral load and positivity rate similar to NPS samples. 
* For most studies: If NPS samples were positive, salvia samples were positive in 70% or more cases. 
* "Salivary samples for SARS-CoV-2 detection was as consistent and sensitive as the nasopharyngeal swabs in most studies, having been effective in detecting asymptomatic infections previously tested negative in nasopharyngeal samples."

#### Summarized Review Mazumder
**Diagnostic tests for SARS-CoV-2: current status and issues**

#### Summary Azzi
Azzi, L., Carcano, G., Gianfagna, F., Grossi, P., Gasperina, D. D., Genoni, A., Fasano, M., Sessa, F., Tettamanti, L., Carinci, F., Maurino, V., Rossi, A., Tagliabue, A., & Baj, A. (2020). **Saliva is a reliable tool to detect SARS-CoV-2. The Journal of infection,** 81(1), e45–e50. https://doi.org/10.1016/j.jinf.2020.04.005
##### Methods
Saliva samples of 25 patients with severe COVID-19 are analyzed by rRT-PCR.
##### Results
* "SARS-CoV-2 was detected in all 25 patients first salivary swab, with different Ct values (range 18.12–32.23, mean value 27.16 +/− 3.07), but all of them were under the Ct value of 33."
* "Interestingly, there was an inverse correlation between the LDH values recorded in the haematochemical analyses and the Ct values, thus the viral load detected in the saliva was correlated to the tissue damage reported by biomarkers (p = 0.04) (Table 2) (Fig. 1a and b)."
* "In contrast, there was not a significant correlation between usRCT and the Ct values (p = 0.07), but an inverse tendency between this inflammatory index and the viral load detected in saliva (Fig. 1c and d) was observed."
* "A striking feature was highlighted in two patients who showed positive salivary results on the same days when their pharyngeal or bronchoalveolar swabs proved to be negative."


#### Summary Opinion by Azzi
**SALIVA IS THE KEY ELEMENT FOR SARS-CoV-2 MASS SCREENING**
Lorenzo Azzi gives arguments for salvia based diagnosis and possible applications such as mass screening. Key points of the [paper by Yokota et al](#summary-yokota-2) are summarized.

#### Summary Berenger
Berenger BM et al., 
**Saliva collected in universal transport media is an effective, simple and high-volume amenable method to detect SARS-CoV-2**, 
Clinical Microbiology and Infection, https://doi.org/10.1016/j.cmi.2020.10.035
##### Methods
* "PCR-positive individuals admitted to the hospital or in the community diagnosed by a NP or OP swab gave their consent"
* Collection Methods:
  1. __Salvia Saline gargle__:
     * "For the saline gargle and spit method, participants took 3 mL of 0.85% sterile saline into their mouth, moved it back and forth to each side of the mouth twice, then gargled for 3 s and spat into a sterile polypropylene urine collection container (~4 cm wide)."
     * "Twenty-nine participants (58.6% hospitalized) had a saline gargle sample collected."
  2. __Salvia UTM method__:
     * "[Pasomsub et al.](#summary-pasomsub) report on a low number of patients (n = 21 positives), finding a sensitivity of 84.2% for saliva collected in UTM compared with NP and oropharyngeal/throat (OP) swabs for the detection of SARS-CoV-2. We therefore decided to test the method of Pasomsub et al. in a larger number of patients." 
     * "For the UTM (universal transport medium) method, participants accumulated saliva for 1 - 2 min (no specific volume) and spat into an empty urine container, and then 3 mL of Copan Italia UTM-RT (Code 330C) was immediately added."
  3. __NP Swap__:
  "A NP swab (Flexible Mini-tip FloqSwab in 3 mL of UTM, Copan) was concomitantly collected from all patients by an infectious disease physician or a paramedic trained in NP swab collections."

##### Results
* ###### Saline Gargle Method
  "Twenty-nine participants (58.6% hospitalized) had a saline gargle sample collected. Using a reference standard of either sample positive (n = 24), the positive agreement (95% confidence interval (CI)) for the saline gargle was 70.8% (50.8-85.1) and NP swab 95.8% (79.8-99.8). In one case, only saliva was positive. The mean time from symptom onset and study collection was 11.6 days (range 3-44)."
  Salvia\Swab     | Swab + | Swab - | Total
  --|--|--|--
  Saliva-Gargle + |     16 |     1  |  17 
  Salvia-Gargle - |      7 |     5  |  12
  Total           |     23 |     6  |  29 

* ###### Salvia-UTM Method
  "Seventy-five patients (9.5% hospitalized) had saliva in UTM collected. The positive agreement for saliva was 84.1% (73.7-90.9) and NP swab 91.3% (82.3-95.9)"
  Salvia\Swab  |   Swab + | Swab - | Total
  --|--|--|--
  Saliva-UTM + |       52 |    6   |  58 
  Salvia-UTM - |       11 |    6   |  17
  Total        |       63 |   12   |  75 


#### Summary Iwasaki
Iwasaki, S., Fujisawa, S., Nakakubo, S., Kamada, K., Yamashita, Y., Fukumoto, T., Sato, K., Oguri, S., Taki, K., Senjo, H., Sugita, J., Hayasaka, K., Konno, S., Nishida, M., & Teshima, T. (2020). **Comparison of SARS-CoV-2 detection in nasopharyngeal swab and saliva.** The Journal of infection, 81(2), e145–e147. https://doi.org/10.1016/j.jinf.2020.05.071
##### Methods
Analyzed the viral nasopharyngeal and saliva samples in 76 patients.
##### Results
* 8 patients were positive for both salvia and nasopharyngeal probes. 1 patient for salvia only and 1 for nasopharyngeal only. 
* A weak tendency for salvia samples to have a higher viral load early and nasopharyngeal swabs in the later phase of Covid-19 was observed. 

Salvia\Swab | Swab + | Swab - | Total
--|--|--|--
Saliva +    |    8 |      1   |  9 
Salvia -    |    1 |     66   | 67
Total       |    9 |     67   | 76 



#### Summary Güclü
Güçlü, E., Koroglu, M., Yürümez, Y., Toptan, H., Kose, E., Güneysu, F., & Karabay, O. (2020). 
**Comparison of saliva and oro-nasopharyngeal swab sample in the molecular diagnosis of COVID-19.** Revista da Associacao Medica Brasileira (1992), 66(8), 1116–1121. https://doi.org/10.1590/1806-9282.66.8.1116
##### Methods 
Three groups of patients are analyzed:
* "Group 1 (30 patients): Hospitalized patients with a finding consistent with COVID-19 in the CT scan of the lung and detected SARS-CoV-2 by PCR in at least one ONS sample."
* "Group 2 (15 patients): Hospitalized patients with a finding compatible with COVID-19 in lung CT examination, but in whom SARS-CoV-2 were not detected in at least two ONS samples by PCR."
* "Group 3 (19 patients): Patients who present to the emergency department with complaints compatible with COVID-19 (fever, cough, shortness of breath) but have normal CT."
##### Results
* Group 1: Saliva samples were positive in 21 (70%) of the 30 patients in Group 1. All ONS samples by group construction. 
* Group 2: Two salvia samples were positive for SARS-CoV-2. No positive ONS samples by group construction.
* Group 3: SARS-CoV-2 was detected in the saliva of two patients, and only in ONS of one patient. 

Results for testees in Group 1 and Group 2 (all patients in group 1 & 2 have Covid with a very high probability):
Salvia\Swab | Swab + | Swab - | Total
--|--|--|--
Saliva +    |    21 |    2    | 23 
Salvia -    |     9 |   13    | 22
Total       |    30 |   15    | 45 


#### Summary Huber
**Large parallel screen of saliva and nasopharyngeal swabs in a test center setting proofs utility of saliva as alternate specimen for SARS-CoV-2 detection by RT-PCR**

##### Methods
* "Adults and children that qualified for a regular SARS-CoV-2 test according to the FOPH (Swiss federal office of public health) and reported to one of the participating test centers or emergency units were enrolled from October 20, 2020 to November 4, 2020. In total 1187 individuals (male 54.8%/female 45.2%) were included (Table1)." 
* "Participants were either asked to clear the throat thoroughly (“Basic”, N = 835) or in an intensified protocol to clear it three times (“Enhanced”, N = 352) and collect about 0.5 – 1 ml of saliva."
* "In the present study, neither eating, drinking nor smoking was controlled as study subjects came for an elective analysis by NPS and thus could only be informed about the saliva sampling on site immediately before the collection."
*  "Immediately after saliva collection, VTM (virus transport medium) was added to the crude saliva and the content mixed through gentle twisting." 
*  "Saliva was collected directly after NPS and both specimens immediately sent for SARS-CoV-2 RT-PCR testing."
##### Results
Salvia\Swab | Swab + | Swab - | Total
--|--|--|--
Saliva +    |    228 |    4   |  232 
Salvia -    |     20 |  935   |  955
Total       |    248 |  939   | 1187 


* In Figure 4 A the measurements points are plotted in a NPS-viral-load to Salvia-viral-load coordinate system. Findings:
  * People experiencing strong symptoms tend to have a higher viral load (lower CT values) both in NPS and salvia. 
  * The NPS viral load is higher for the majority of testees.
  * The viral load in salvia and NPS differs a lot for some testees.
* Figure 4 B shows that earlier after symptom onset (only symptomatic patients) patients get tested, the higher the viral load is. 



#### Summary Matic
**Practical challenges to the clinical implementation of saliva for SARS-CoV-2 detection**
While the experiments described are interesting, the introduction section of their paper is highly recommended: it provides a concise overview of the different diagnostic methods and associated challenges.
Experiments:
1. Comparison of salvia and NPS for SARS-CoV-2 detection
2. Whether delayed analysis of salvia samples reduces detection rates. 

##### Methods
1. Experiment __Comparing NPS and Salvia Diagnosis:__
     * Testees: "From March–May 2020, paired saliva and nasopharyngeal swabs were received from patients under investigation for COVID-19 from various clinical settings: inpatients, residents of long-term care (LTC) facilities, healthcare workers, and out- patients."
     * NPS Collection: "Nasopharyngeal swabs were collected using COPAN Flocked Swabs with 2 mL of Universal Transport MediumTM (COPAN, Brescia, Italy) or 3 mL of BDTM Universal Viral Transport System (Becton Dickinson, Sparks, MD), depending on availability."
     * Salvia Collection: "After the nasopharyngeal swab was collected, patients were asked to provide approximately 1 mL of saliva by pooling saliva in the mouth and spitting into a sterile screw-top container (Starplex Scientific Inc., Etobicoke, Canada)."
2. Experiment __Stability of SARS-CoV-2 RNA for Detection__
  Five salvia samples (2 from patients and 3 from volunteers spiked with known positive NPS samples) were stored at room temperature and analyzed at delayed time points: 0, 12, 24, 36, and 48 h.

##### Results
1. Experiment __Comparing NPS and Salvia Diagnosis:__
  
    Salvia\Swab | Swab + | Swab - | Total
    --|--|--|--
    Saliva +    |     15 |     1  | 16 
    Salvia -    |      6 |    52  | 58
    Total       |     21 |    53  | 74 

2. Experiment __Stability of SARS-CoV-2 RNA for Detection__
  The CT values of the E-gene in the salvia samples stayed constant over the time points measured. 




#### Summary Pasomsub
[in work]
**Saliva sample as a non-invasive specimen for the diagnosis of coronavirus disease 2019: a cross-sectional study**
Pasomsub E et al., Saliva sample as a non-invasive specimen for the diagnosis of coronavirus disease 2019: a cross- sectional study, Clinical Microbiology and Infection, https://doi.org/10.1016/j.cmi.2020.05.001
##### Methods
"From 27 March to 4 April 2020, we prospectively collected saliva samples and a standard nasopharyngeal and throat swab in persons seeking care at an acute respiratory infection clinic in a university hospital during the outbreak of COVID-19. Real-time polymerase chain reaction (RT-PCR) was performed, and the results of the two specimens were compared."
##### Results
Salvia\NPS | NPS + | NPS - | Total
--|--|--|--
Saliva +   |    16 |   2   |   18 
Salvia -   |     3 | 179   |  182
Total      |    19 | 181   |  200 


#### Summary Procop
Procop GW, Shrestha NK, Vogel S, Van Sickle K, Harrington S, Rhoads DD, Rubin BP, Terpeluk P. 2020. 
**A direct comparison of enhanced saliva to nasopharyngeal swab for the detection of SARS-CoV-2 in symptomatic patients.** 
J Clin Microbiol 58:e01946-20. [https:// doi.org/10.1128/JCM.01946-20]()
##### Methods
"An enhanced saliva specimen (i.e., strong sniff, elicited cough, and collection of saliva/secretions) was collected without transport medium prior to collection of NPS from 224 patients with symptoms deemed consistent with COVID-19."
##### Results
216 of the 224 patients had analyzed both samples without errors (9 errors in total).

Salvia\Swab     | Swab + | Swab - | Total
--|--|--|--
Saliva +        | 38     | 1      | 39 
Salvia -        |  0     | 177    | 177
Total           | 38     | 178    | 216 

###### Notes
* "The overall mean (SD) CT value for the positive NPS specimens was 20.55 (5.36) cycles, whereas the corresponding overall mean (SD) CT value for enhanced saliva specimens was 24.16 (4.80) cycles, for a mean difference in CT value for paired NPS and saliva specimens of -3.61 (95% CI, -5.78 to -1.44; P = 0.002)."
* "There was little correlation between the CT values (i.e., viral loads) of the positive saliva and NPS specimens (Pearson correlation coefficient, r = 0.162; P = 0.333) (Fig. 1C)."




#### Summary Rao
Rao M, Rashid FA, Sabri FSAH, et al. 
**Comparing nasopharyngeal swab and early morning saliva for the identification of SARS-CoV-2.** 
Clinical Infectious Diseases : an Official Publication of the Infectious Diseases Society of America. 2020 Aug. 
DOI: [https://doi.org/10.1093/cid/ciaa1156]()
##### Methods
* Participants:
  "This prospective single center diagnostic study was conducted among 217 individuals who were tested positive for SARS- CoV-2 via NPS at a COVID-19 quarantine center, MAEPS. These selected individuals were on days 8–10 of isolation during the sampling."..."Individuals with respiratory aid were excluded."
  "All participants were male recruits and asymptomatic at the time of sampling. The median age of the participant was 27 (IQR: 18–36) years." 
* Salvia Collection:
  "Before collecting swabs, individuals were asked to provide self-collected deep throat saliva sample in a sterile collection container. Instruction on self-collected deep throat saliva was announced to participants a day prior. Briefly, upon waking up, the individuals were instructed to avoid food, water, and brushing of teeth before the collection of 2 mL of saliva."
* Swab Collection:
  "As a standard protocol, NPS from individuals were collected using sterile flocked swab and placed in sterile tube containing viral transport medium (VTM)."
* Sample Processing
  * "All samples were stored at room temperature and transported to research lab at Institute for Medical Research, Kuala Lumpur, within 5 hours of sample collection for further processing."
  * In brief: Samples were inactivated at 65 C for 1 hour, nucleic acid extraction by MagNA Pure 96 system and Viral NA Small Volume extraction kit (Roche Diagnostic GmBH, Germany), from 200 μL of viral transport medium containing the NPS or 200 μL of saliva. Testing with 1-step RT-PCR of Real-Q 2019 nCoV detection kit (Biosewoom, Inc, South Korea), which detects the E-gene and RdRp of SARS-CoV-2 and human RNase gene as an internal control. 
  * "Samples were classified as positive for SARS-CoV-2 when both E-gene and RdRp primer-probe sets were detected at cycle threshold (Ct) value of <38."
##### Results
* "Among the 217 eligible participants, 73.7% (160/217) participants were tested positive for SARS-CoV-2 either from saliva, NPS, or both. There was an overall significant fair agreement between saliva and NPS (73.7%; 160/217, κ 0.260, 95% CI .158 – .363, P < .001). Among patients with concordant results, 45.6% (73/160) had virus detected in both saliva and NPS. Eighty-seven patients had discordant results between saliva and NPS via SARS-CoV-2 RT-PCR assay, by which 47.5% (76/160) patients with virus detected in saliva but not in NPS and 6.9% (11/160) patients with virus detected in NPS but not in saliva."
* Saliva shares large distribution of Ct value in comparison to NPS. Meanwhile, most NPS Ct values were edging the upper limit (Figure 2)."

Salvia\Swab     |  NPS + |  NPS - | Total
--|--|--|--
Saliva +        |  73    |    76  |  149
Salvia -        |  11    |    57  |   68
Total           |  84    |   133  |  217



#### Summary Wang
**Detection of SARS-CoV-2 in Different Types of Clinical Specimens**
##### Methods
* "We investigated the biodistribution of SARS-CoV-2 among different tissues of inpatients with coronavirus disease 2019 (COVID-19) diagnosed based on symptoms and radiology and confirmed by SARS-CoV-2 detection."
* "Most of the patients presented with fever, dry cough, and fatigue; 19% of patients had severe illness." 
* "A cycle threshold value less than 40 is interpreted as positive for SARS-CoV-2 RNA." 
* "Four SARS-CoV-2 positive fecal specimens with high copy numbers were cultured, and then electron microscopy was performed to detect live virus."
##### Results
* Positive rates in decreasing order:
  * Bronchoalveolar lavage fluid (14 of 15; 93%)
  * Sputum (72 of 104; 72%)
  * Nasal swabs (5 of 8; 63%)
  * Fibrobronchoscope brush biopsy (6 of 13; 46%)
  * Pharyngeal swabs (126 of 398; 32%)
  * Feces (44 of 153; 29%)
  * Blood (3 of 307; 1%)
  * Urine specimens (0 of 72; 0%) 

* Two of the four cultured fecel samples showed signs of virus replication.



#### Summary Williams
**Saliva as a non-invasive specimen for detection of SARS-CoV-2**
##### Methods
* Between 25th March and 1st April 2020 in Melbourne, 622 patients provided NPS samples, 522 of those 622 patients provided additionally saliva samples for Covid-19 tests. 
* Collection Methods:
  Salvia: Patients were asked to pool saliva in their mouth for 1-2 minutes prior to collection.
##### Results
* 39 of 622 (6.3%) patients had PCR-positive NPS, and 33 of those 39 NPS positive (84.6%) had SARS-CoV-2 detected in saliva.
* 1 positive of 50 (2%) salvia sample of patients whose NPS sample was negative (only the salvia of 50 patiens whose NPS sample was negative was analyzed).
##### Limitations
Data not uniform/consistent:
* Of the 622 patients with NPS samples only 522 provided salvia samples (but not written if the positive 39 belonged to the 522).
* Only 50 salvia samples of those tested negative by NPS were analyzed.  



#### Summary Vaz
**Saliva is a reliable, non-invasive specimen for SARS-CoV-2 detection**
##### Methods
155 Participants with signs/symptoms suggesting SARS-CoV-2 infection underwent a nasopharyngeal swab (NPS) and/or oropharyngeal swab (OPS) and saliva collection in Salvador, Brazil. 
##### Results
Salvia\NPS or OPS | NPS/OPS + | NPS/OPS - | Total
--|--|--|--
Saliva +         |    67      |       2   |  69 
Salvia -         |     4      |      82   |  86 
Total            |    71      |      84   | 155 


#### Summary Yokota 1
**Equivalent SARS-CoV-2 viral loads between nasopharyngeal swab and saliva in symptomatic patients**
##### Methods
* "Forty-two patients diagnosed with COVID-19 by positive qRT-PCR of NPS were enrolled in this study."
* "Paired NPS and saliva samples were simultaneously collected from all patients upon hospital admission between June 12, 2020 and August 6, 2020."
* "Self-collected saliva was diluted 4-fold with phosphate buffered saline and centrifuged at 2000 × g for 5 min to remove cells and debris. RNA was extracted from 200 μL of the supernatant or nasopharyngeal swab samples using QIAsymphony DSP Virus/Pathogen kit and QIAamp Viral RNA Mini Kit (QIAGEN, Hilden, Germany)."
##### Results
Salvia\Swab | NPS + | NPS - | Total
--|--|--|--
Saliva +    |    34 | 4     | 38 
Salvia -    |     0 | 4     | 4
Total       |    34 | 8     | 42 

#### Summary Yokota 2
**Mass screening of asymptomatic persons for SARS-CoV-2 using saliva**
Yokota, I., Shane, P. Y., Okada, K., Unoki, Y., Yang, Y., Inao, T., Sakamaki, K., Iwasaki, S., Hayasaka, K., Sugita, J., Nishida, M., Fujisawa, S., & Teshima, T. (2020). Mass screening of asymptomatic persons for SARS-CoV-2 using saliva. Clinical infectious diseases : an official publication of the Infectious Diseases Society of America, ciaa1388. Advance online publication. https://doi.org/10.1093/cid/ciaa1388
##### Methods
Individuals from two cohorts of asymptomatic persons: the contact tracing cohort and the airport quarantine cohort had specimens from nasopharyngeal swabs (NPS) and saliva samples analyzed by PCR.
##### Results
* "In this mass-screening study including 1,924 individuals, the sensitivity of nucleic acid amplification testing with nasopharyngeal and saliva specimens were 86% (90%CI:77-93%) and 92% (90%CI:83-97%), respectively, with specificities greater than 99.9%. The true concordance probability between the nasopharyngeal and saliva tests was estimated at 0.998 (90%CI:0.996-0.999) on the estimated airport prevalence at 0.3%. In positive individuals, viral load was highly correlated between NPS and saliva."
* "Both nasopharyngeal and saliva specimens had high sensitivity and specificity. Self- collected saliva is a valuable specimen to detect SARS-CoV-2 in mass screening of asymptomatic persons."

###### Contact Tracing Cohort
Salvia\Swab | NPS + | NPS - | Total
--|--|--|--
Saliva +    | 38    |   6   |  44 
Salvia -    |  3    | 114   | 117
Total       | 41    | 120   | 161

###### Airport Quarantine cohort
Salvia\Swab | NPS + | NPS - | Total
--|--|--|--
Saliva +    |  4    |    0  | 4 
Salvia -    |  1    | 1758  | 1759
Total       |  5    | 1758  | 1763 

### References RNA Detection Stability

#### Summary Ott
**Simply saliva: stability of SARS-CoV-2 detection negates the need for expensive collection devices**
##### Methods
* "RNA was extracted from saliva samples collected from COVID-19 inpatients and healthcare workers at the Yale-New Haven Hospital(...)"
* Experiment 1: "RNA templates were tested by RT-qPCR for SARS- CoV-2 RNA (N1) on day of collection (~12 hours post collection) and at various time points after the storage of the remaining, unsupplemented samples at temperatures of -80°C, -20°C, +4°C, 19oC, or 30°C."
* Experiment 2: Cell Culture Test: Vero-E6 cells were inoculated with salvia samples. 
##### Results
* Experiment 1 __Detection Statiblity__
   * Supplemental Figure 1: The Ct values of the viral RNA sequence for the N1 gene was not much different after storing at the different conditions (- 80°C freeze, +4°C, room temperature  and 30°C) for several days. 
   * "Supplemental Figure 2. Detection of human RNAse P (RP) declines over time when stored in saliva in warmer conditions."
* Experiment 2 __Infectable Virions__
   Unclear if virions replicated: For some cultures a lower Ct value was observed (indicating replication), however follow up checking by plague assays was negative.



### Time Series



#### Summary To
**Temporal profiles of viral load in posterior oropharyngeal saliva samples and serum antibody responses during infection by SARS-CoV-2: an observational cohort study**
[in work]

#### Summary Wyllie
**Saliva is more sensitive for SARS-CoV-2 detection in COVID-19 patients than nasopharyngeal swabs**
##### Methods
* Participants: 
  * 44 COVID-19 inpatients and 29 of those 44 had paired samples taken (Table 1). Initial Covid-19 diagnosis at hospital admission by NPS for all patients. 
  * "Asymptomatic healthcare workers (e.g., without fever or respiratory symptoms) with occupational exposure to patients with COVID-19 were invited to enroll in the study."
* Sample Collection: 
  * Salvia: "Saliva samples were self-collected by the patient. Upon waking, patients were asked to avoid food, water and brushing of teeth until the sample was collected. Patients were asked to repeatedly spit into a sterile urine cup until roughly a third full of liquid (excluding bubbles), before securely closing it."
  * NPS: "Nasopharyngeal samples were taken by registered nurses using the BD universal viral transport (UVT) system. The flexible, mini-tip swab was passed through the patient's nostril until the posterior nasopharynx was reached, left in place for several seconds to absorb secretions then slowly removed while rotating."
* Sample Handling: "All samples were stored at room temperature and transported to the research lab at the Yale School of Public Health within 5 hours of sample collection."
  
##### Results
* In Figure 1 in their paper the viral loads from salvia and NPS are compared. The mean viral load in salvia is about 5 times higher than in NPS. In Figure 1c the paired viral load points are plotted. The plot corresponds to the following table: 

  Salvia\NPS       | NPS +  | NPS - | Total
  --|--|--|--
  Saliva +         |   24   |    8  | 32
  Salvia -         |    3   |    3  |  6 
  Total            |   27   |   11  | 38 

* In Figure 2 the measurements of viral titers are shown as time series per patient. Not only are more salvia samples positive compared to NPS as noted above, but also the viral load is less variable i.e. the decrease of the viral load in salvia is more monoton.


#### Wyllie Letter
**Saliva or Nasopharyngeal Swab Specimens for Detection of SARS-CoV-2**
This is the letter version of the investigation by [Wyllie et al](#summary-wyllie).

> For a deeper dive, I recommend the [full version](#summary-wyllie) since crucial details such as collection time (morning before breakfast) are left out in letter version. The letter is useful for an overview though and was the initial motivation for me to read about salvia diagnosis of Covid-19.


#### Summary Yang
**Laboratory Diagnosis and Monitoring the Viral Shedding of SARS-CoV-2 Infection**
[https://doi.org/10.1016/j.xinn.2020.100061]()
##### Methods
* "Four hundred and ten patients with confirmed COVID-19 from the Guangdong CDC (Center for Disease Control and Prevention), who were hospitalized in Shenzhen Third People’s Hospital, were included."
* "3,552 respiratory specimens from 410 patients were serially collected during hospitalization, including 559 oropharyngeal swabs, 2,231 nasopharyngeal swabs, 696 sputum samples, and 66 BALF samples."
* "The different types of specimens from laboratory-confirmed COVID-19 cases were stratified into three groups based on the collection times: 0–7, 8–14, and R15 d.a.o."


##### Results
* Table 1 shows that NPS have quite a low dection rate (only about 60% for mild cases) and sputum has the highest detection rate (about 80%). Results for the first 7 days (shedding of infectious virions highest in first week[citation in work]):
  ###### Positive rate (n/N, %) in the first 7 days of illness
  Specimen      | Severe Cases | Mild Cases
  --|--|--
  oropharyngeal | 40/55 (72.7) | 84/158 (53.2)
  nasopharyngeal| 58/68 (85.3) | 195/314 (62.1)
  sputum        | 14/16 (87.5) | 38/46 (82.6)
  BALF          | 2/2 (100.0)  |0/0 (0)

  ###### Ct values (median, range) in the first 7 days of illness__
  Specimen      | Severe Cases | Mild Cases
  --|--|--
  oropharyngeal | 29.5 (18-36) | 29.5 (15-37)
  nasopharyngeal| 29.3 (19-38) | 29 (15-38)
  sputum        | 26 (19-32)   | 28 (18-38)
  BALF          | 22.5 (21-24) | -

* Figure 2 shows time series of viral loads for a series 21 individuals (18 severe and 3 mild patients). Key findings are: 
  * While *no* viral RNA was detected in the upper respiratory tract, at the *same* time it was positive in BALF for the cases 01, 03, 07, 11, 12, 13, and 16 (Figure 2).  
  * "Viral RNA was tested negative in the first three or all of the upper respiratory samples. However, viral RNA was detected either by the further detection of upper respiratory samples (cases 21–28) or BALF samples (cases 02, 04, 06, and 07) (Figures 2 and S1)." 
* Figure 3 shows CT Scans of the mentioned 11 cases (02, 04, 06, 07 and 21-28) tested negative at least 3 time in the upper respiratory tract. The CT  show typical ground-glass opacity in the lungs, suggesting a viral pneumonia.



#### Summary Zhou
**SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients**
#### Methods
"We analyzed the viral load in nasal and throat swabs obtained from the 17 symptomatic patients in relation to day of onset of any symptoms (Fig. 1C)." 
#### Results
* "Higher viral loads (inversely related to Ct value) were detected soon after symptom onset, with higher viral loads detected in the nose than in the throat." 
* "Our analysis suggests that the viral nucleic acid shedding pattern of patients infected with SARS-CoV-2 resembles that of patients with influenza4 and appears different from that seen in patients infected with SARS-CoV."
* "The viral load that was detected in the asymptomatic patient was similar to that in the symptomatic patients, which suggests the transmission potential of asymptomatic or minimally symptomatic patients."
* For many patients the time series of their nasal and throat swabs are varying and don't have single local maxima but several. The viral load can even be undetectable one day and be positive the next day. //=> to exclude infection several negative samples are needed.




## References

* #### Ioannidis
  Ioannidis J. **The infection fatality rate of COVID-19 inferred from seroprevalence data.**
  accessed version published on 14.10.2020: https://www.who.int/bulletin/online_first/BLT.20.265892.pdf



## Appendix
> #### Personal Covid Experience
> I likely had Covid-19 myself in early to mid March 2020 when temperatures were still about -10 degree Celsius in my area. For me the initial experience was very different from a common respiratory disease and so diagnosis was not obvious, at least back then: 1) lack of respiratory symptoms - not a single cough if I recall correctly -  2) the supposed low prior probability according to the official prevalence and 3) because I didn't want to have it and so was satisfied with not experiencing known and obvious symptoms. It first felt like a minor food poisoning but no vomiting or the like. Just felt strange for about two days. Soon these initial symptoms faded and I felt fit again. I went on doing some sport session. The sport session went well by itself, but it turned out to have been a bad idea: The following days some chills started and accompanied by a temperature increase, up to 36.x from my normal 35 degrees. But since I didn't record the temperature systematically before, the temperature measurements of 36.x degrees were calming instead of ringing the bells. Later the temperature further increased to 37.x accompanied by a strange sticky feeling in the chest; at this point I started to consider Covid as likely.
> Counting everything up it likely was Covid-19 and the sports session likely gave the viruses the opportunity to spread within the lungs. Noteworthy too: The day before the first symptoms was long and rather stressful, so the immune system likely wasn't prepared well to control the viruses initially. But it then reacted timely: The initial  strange feeling started less than twelve hours after the likely infection event (somebody coughing about 4 meters away in the same room and I was there for about 1 hour). 
