
Diagnosis and Viral Load
========================

* [Summary](#summary)
  * [Covid-19 Diagnosis](#covid-19-diagnosis)
  * [SARS-CoV-2 Viral Load](#sars-cov-2-viral-load)
  * [Take Away](#take-away)
* [Introduction to Diagnosis](#introduction-to-diagnosis)
* [History of Covid-19 Diagnosis](#history-of-covid-19-diagnosis)
* [Methods to Directly Detect Viruses](#methods-to-directly-detect-viruses)
  * [Measure the Viral Load](#measure-the-viral-load)
  * [Detect Infectious Virions](#detect-infectious-virions)
  * [Detect Metabolism of Viruses](#detect-metabolism-of-viruses)
* [Methods to Indirectly Detect Viruses](#methods-to-indirectly-detect-viruses)
  * [Detect Tissue damage](#detect-tissue-damage)
  * [Detect Immune Response](#detect-immune-response)
* [Taking Specimens](#taking-specimens)
  * [Locations of Virions](#locations-of-virions)
  * [Collection Timing](#collection-timing)
  * [Collection Methods](#collection-methods)
* [Introduction to Viral Load](#introduction-to-viral-load)
* [Diagnosis and Viral Load at Different Locations](#diagnosis-and-viral-load-at-different-locations)
  * [Literature for Viral Load and Diagnosis](#literature-for-viral-load-and-diagnosis)
  * [Diagnosis in the Nose](#diagnosis-in-the-nose)
  * [Diagnosis in the Mouth](#diagnosis-in-the-mouth)
  * [Diagnosis in the Lungs](#diagnosis-in-the-lungs)
  * [Challenges regarding Viral Load and Diagnosis](#challenges-regarding-viral-load-and-diagnosis)
* [Saliva Diagnosis](#saliva-diagnosis)
  * [Literature on Saliva Diagnosis](#literature-on-saliva-diagnosis)
  * [Viral Load for Different Saliva Sampling Methods](#viral-load-for-different-saliva-sampling-methods)
  * [Notes on Saliva Diagnosis](#notes-on-saliva-diagnosis)
* [Summarized References](#summarized-references)
  * [Viral Load in Sputum](#viral-load-in-sputum)
  * [Viral Load in Saliva Summarized Refs](#viral-load-in-saliva-summarized-refs)
  * [References RNA Detection Stability](#references-rna-detection-stability)
  * [Time Series of Viral Load in Saliva](#time-series-of-viral-load-in-saliva)
  * [Time Series Viral Load in Multiple Locations](#time-series-viral-load-in-multiple-locations)
  * [Viral Load Nose and Throat Series](#viral-load-nose-and-throat-series)
  * [Flow Patterns of Body Fluids](#flow-patterns-of-body-fluids)
* [Appendix](#appendix)
  * [Calculation of Viral Load from CT_Value](#calculation-of-viral-load-from-ct_value)


**entire page is in work**

## Summary
Methods and locations to diagnose Covid-19 are discussed. A focus is on methods detecting the SARS-CoV-2 viral load at different locations. For this reason diagnosis and viral load are in the same chapter.

### Covid-19 Diagnosis
* The [viral load](#viral-load) such as antigen test or PCR tests. PCR test offer a little better sensitivity and specificity but the sensitivity depends mainly on the location.
* Detect infections indirectly e.g. visual changes in the lungs by X-ray images. Detection only if the lung is notable infected but high sensitivity in this cases. 
* Antibodies which are part of the immune response. High specificity. Very low sensitivity in the early phase since antibodies are not yet produced. Higher sensitivity later on since most people produce antibodies. 

### SARS-CoV-2 Viral Load
The viral load measures the amount of virions present. Whether a viral load can be detected (=detection rate) and how high it is (=viral load), is dependent on the location, the sampling method and the infected locations/tissues of a patient. SARS-CoV-2 can infect many locations, the infected locations vary between patients and virions are distributed with fluids e.g. the air and the mucus in different parts of the respiratory tract. Some locations are easy to sample such as saliva or the nose swabs but detection rates are low (depending on the method and the infection they vary between 40% and 80%). Other methods are more cumbersome to sample such as nasopharyngeal aspirate, sputum or bronchoalveolar lavage fluid but offer higher sensitivities.

* The standard method NPS analyzed by PCR has a detection rate of about  50 - 70 % across infections and the viral loads measured vary. 
* Morning before eat saliva analyzed by PCR has a higher detection rate than NPS in the first days of symptom onset (about 10% higher, so 60% - 80%). The viral load and thus the detection rate is reduced if the salivary flow increases e.g. eating or chewing gums.
* Morning sputum and nasopharyngeal aspirate have high viral load and good detection rates. 
* A high viral load in the sputum or in saliva is an indication for severe disease progression. The viral load in NPS samples isn't an indicator. 

### Take Away
* __Recommended Diagnosis__: Morning sputum, repeated morning saliva have a high viral load and good detection rates and predict the disease severity. 
* __With Reservation__: The standard methods nasopharyngeal swabs (NPS) has only a detection rate of about 60%. Nasopharyngeal aspirate has a good detection rate but is not so easy to sample. CTs of the lungs work to detect lung infections causing tissue damage (possible super spreaders) but there's radiation. 


## Introduction to Diagnosis

> ###### Diagnose a Disease
> Figuring out the causes of the disease.

A defined in the [Covid-19 chapter](./covid19.md) a [disease](./covid19.md#disease) is a disordering of the body. The disordering and induced immune responses can be inspected across all causes of a disease (section [Methods to Indirectly Detect Viruses](#methods-to-indirectly-detect-viruses). For virus diseases the viruses can be detected directly (section [Methods to Directly Detect Viruses](#methods-to-directly-detect-viruses)).

## History of Covid-19 Diagnosis
* In late 2019 and January 2020 Covid-19 was mostly diagnosed based on lung CT findings.
* In February 2020 PCR test became more widely available and often specimens from the lower respiratory tract such as BALF or sputum were used. 
* From February/March 2020 to December 2020 the standard SARS-CoV-2 tests are nasopharyngeal swabs in most locations of the world. The viral load is detected either by [PCR-Test](#pcr-test) or [Antigen-Tests](#antigen-test).

  > In the very early phase of the Covid-19 pandemics PCR test were scarce and antigen tests not available so the work for BALF/sputum sampling was not the bottleneck. In the next months PCR test became more widely available but still limited, nasopharyngeal swabs were used: NPS sampling was doable since testing capacity was still limited. Everybody then just kept on doing NPS even there have been early studies pointing to the benefits of other methods such as self collected saliva. 
* Late 2020 antigen tests are getting more common.
* 2021? Alternate Locations? Rethink [Diagnosis Locations and Methods](#diagnosis-locations) and a chapter is devoted to [saliva based diagnosis](#saliva-diagnosis).



## Methods to Directly Detect Viruses

### Measure the Viral Load

In tissues containing infected cells or in the lining fluids protecting/cleaning these tissues the viral load can be measured by detecting building blocks.

#### Genome Detection
Test for specific sequences in the genome of virions (positive single stranded RNA for coronaviruses). The amount of virions (called [viral load](#viral-load)) - can be determined with high specificity and sensitivity. Examples are given in the section [Common Diagnosis Methods](#common-diagnosis-methods).

> ###### PCR Test
> A *PCR test* is a method to detect specific sequences of RNA or DNA. PCR doubles specific DNA/RNA strands in cycles called CT cycles. The concentration of the DNA/RNA double each round, at some point it is high enough to be detected by some helper method e.g. by optical fluorescence. The doubling cycles needed are called CT values.

##### From CT-value to Concentration
As described the PCR method approximately doubles the viral_load each cycle until detectable by the helper method. To calculate to original viral load, one needs to undo the doubling-cycles. Thus a difference of 10 CT-cycles corresponds to an amplification of 2^10=1024: If 10 additional CT cycles are needed the starting viral load was 1024 times smaller ([Calculation](#calculation-of-viral-load-from-ct_value) in the Appendix).

##### PCR Detection Limit
In theory a single sequence of RNA/DNA can be amplified. Practically PCR apparatuses have some detection limit often this is in the order of a few thousand copies per milliliter. This correspond to a maximal PCR cycle number (often 40). 


#### Protein Detection
Methods detecting proteins of virions are usually called antigen tests (virus proteins are called antigens since these proteins can be recognized by the immune system as harmful).
* There are many different products of antigen tests which vary a lot in sensitivity and specificity.
* The sensitivity and specificity of antigen tests tend to be lower than PCR based tests for most antigen test kits available as of December 2020.

> ###### Antigen Test
> Proteins can act as antigens and there are ways to produce antibodies which bind to them. Often these tests give only binary results.


### Detect Infectious Virions
[in work: to be checked and incomplete]
Samples possibly containing virions can be put on cell or organ cultures. These cell or organ cultures then can be inspected by another methods such as detecting tissue change/damage or detecting RNA to see whether there were infectious virions in the  samples. Virus and especially coronavirus culturing is difficult, therefore for routine diagnosis virus culturing is not suitable. However virus culturing is very useful for research purposes: e.g. to detect whether and how long infected individuals shed infectious virions. 
##### Cell Cultures
##### Organ Cultures
##### Plague Assays
The samples can be diluted to the point new virus colonies are originating from single virions can be distinguished. In this way the number of infectious virions can be determined. 


### Detect Metabolism of Viruses
Coronaviruses produce temporary RNA similar to mRNA of DNA based life forms can be detected by [PCR-Tests](#pcr-test). This RNA is *not* packaged into virions and therefore is degraded quickly. Because of the fast degradation, detecting temporary RNA means there are viruses with a running metabolism. 



## Methods to Indirectly Detect Viruses

### Detect Tissue damage
Detecting tissue damage is generally not very specific since other diseases e.g. other virus infections cause similar tissue damage.
#### Biochemical 
E.g. detecting cell damage through cell decay products e.g. distinctive proteins. Often useful in combination with other methods: if there's an increased SARS-2 viral load at a location combined with increased cell decay products then SARS-2 likely replicates at this location. 
#### Optical 
E.g. CT (quite accurate but high radiation to sensible locations) or Xray (not very accurate some radiation): round glassy occupancies in the lung are characteristic for Covid-19. Other (viral) lung infections look similar so the specificity depends on the prevalence of SARS-CoV-2 compared other virus infections. 

### Detect Immune Response
#### Antibody Test
[in work] 
For infections triggering certain alarms, the immune response involves antibodies which can be detected in the blood. The antibodies are in the serum of the blood. Different kinds of antibodies are produced at different stages during and after an infection. Only infections in later stages and past infections can be diagnosed by this method since the viruses replicate first to high enough titers to trigger the alarms and then a couple of days are needed to produce antibodies. 

> ###### Serology
> Methods analyzing the blood serum are called *serology*. Antibody detection is referred as serological (Covid) tests. 



## Taking Specimens

### Locations of Virions
Two types of locations are suitable for diagnosis: 
* Tissues with active viral replication. The tissues can either be sampled by a biopsy or by washing them and sucking the cleaning water which is called [aspiration](#aspirate) in medicine or just by swabbing the outer cell layer.
* Viral Load in body fluids ([Viral Load](#viral-load)) from virus replication in close tissues or deposition. Sampled by swab or aspiration. 

### Collection Timing
The human metabolisms follows a daily=circadian (circa = about, dies=day) cycle. This true also for many fluids which can be sampled for disease diagnosis e.g. for the sputum and saliva. 

#### Saliva Collection Timing
The saliva is secreted from different glands in the mouth. How much each of these glands secrets depends on activities such as sleeping or eating (very readable described in [Dawes](#summary-dawes)). The saliva flow rate, location and source yields different microhabitats for bacteria and viruses. [Carlson-Jones et al](#summary-carlson-jones) investigated these habitats with flow cytometry and observe that both the bacteria and virus like particles concentrations peak after sleep.
This morning peak is also observed for SARS-CoV-2: [Hung et al](#summary-hung)

### Collection Methods
> ###### Aspirate
> An *aspirate* is a fluid that is collected by suction (aspirare is latin for to breath).

> ###### Swab
> A *swab* is an absorbent pad or piece of material used in medicine e.g. to collect mucus specimens. 

> ###### Biopsy
> A *biopsy* is a tissue sample taken for the diagnostic purposes. 

> ###### Autopsy
> *Autopsies* are tissue specimens taken post mortem. They are useful for the analysis of the viral load in different organs.

> ###### Sputum Collection
> *Sputum collection* is a special method exploiting the tickling sensitivity of the throat: A saline solution is inhaled provoking a cough (in most individuals) which is captured. 




## Introduction to Viral Load 
[**in work and unfinished**]

> ##### Viral Load
> The *viral load* denotes how many virions are present in specimen of tissue or body fluid.

A viral load is observed in locations playing different roles in an infection:
* __Primary infection locations__: the viruses replicate to high numbers and spread to other locations inside the organism and other organisms. Mainly the lungs, to a lesser extent the mouth e.g. salivary glands. 
* __Minor infection locations__: Deposited virions can infect local cells and thus the viral load is amplified. However these locations play only a minor role in the disease and transmission of the viruses. In the nose and the mouth are susceptible cells amplifying the viral load. 
* __Deposit only locations__: Virions are deposited but do *not* replicate: They are either inactivated or there are no susceptible cells. E.g. fluids from the respiratory tract are cleared towards the stomach where the virions are destroyed by the stomach acid. Therefore in the stomach, respiratory parasites can be detected but usually they do not infect the stomach. Deposit only locations are not reliable for diagnosis however, since the viral load is to low to be detected.

Sampling the **surfaces of tissues** is enough and biopsies are mostly not necessary: 
* __Viral Load on Tissue Surfaces:__ Coronaviruses have the habit of releasing their virions of the apical side of cells in epithelium tissues ([Orientation Matters](./coronavirus.md#orientation-matters)). The surfaces of tissues are often covered by mucus or fluids which can be collected either by [swabs](#swab) or by [aspiration](#aspirate). If the tissues surfaces are not covered the surfaces can be washed with an appropriate fluid and collected by aspiration. 
* __Viral Load in Tissues__ Tissues biopsies/autopsies show how much virions penetrate inside tissues and organs.

#### Relevance of Viral Load
Knowing the viral load observed in different fluids and tissues is relevant for:
* __Diagnosis__: For an optimal diagnosis location, the viral load is detectable across the different disease patterns caused by SARS-CoV-2 and taking specimens is easy.
* __Transmission__: Knowing the viral load in different locations helps to understand and prevent transmissions of SARS-CoV-2 virions.
* __Understanding Covid-19__: Knowing at which stages which locations are infected helps to predict the disease. Which is important to predict possible damage and to do the right treatment. 


## Diagnosis and Viral Load at Different Locations
* For accessibility reasons investigating the upper respiratory is preferred. The viruses cause few tissue damage so sensitive methods such as PCR test/Antigen tests are used to analyze the nose mucus or saliva.   
* The largest number of susceptible cells to SARS-CoV-2 are in the lower respiratory tract. Tissue damage can be observed there by imaging (CT/Xray) and the viral load measured by analyzing sputum, endotracheal aspirate or bronchoalveolar lavage. 

### Literature for Viral Load and Diagnosis
* A recommended review for the viral load in upper respiratory tract specimens is [Lee et al](#summary-review-lee).
* [Yang et al](#summary-yang) investigated the viral load across locations the respiratory tract.

### Diagnosis in the Nose
#### Nasopharyngeal Swabs
Nasopharyngeal Swabs (NPS) sample posterior nasopharyngeal mucus and are, as of December 2020, often used as 'gold standard' for Covid-19 diagnosis. NPS samples can be analyzed either by PCR or antigen tests.
* quite easy to take but trained persons are needed
* the sensitivity is 
  * 85% for severe cases and 62 % for mild cases in first week of illness. In the second week 69% for severe and 53 % for mild cases ([Yang](#summary-yang))
  * 71% in the first 5 days of hospitalization [Wyllie](#summary-wyllie-letter) //patients initially positive by NPS so results biased and the 71% are likely to high (the study is about comparing saliva and NPS)
#### Nasopharyngeal Aspirate 
[in work] As NPS nasopharyngeal aspirates samples the back of the nose but from a much larger area. The large area reduces the randomness and the detection rate is superior [citations in work]. 

#### Anterior Nose Mucus
Detection rates tend to be even lower than NPS. In the review of  [Lee et al](#summarized-review-lee) some studies investigating the anterior nose mucus are cited. 

### Diagnosis in the Mouth
Varies across the methods. Early morning saliva has high viral loads and good detection rates as described in [Saliva Sampling Procedures](#saliva-sampling-procedures) in the section [saliva based Diagnosis](#saliva-diagnosis). 

> Separate section since I consider saliva diagnosis as the best option for routine Covid diagnosis. 

#### Oropharyngeal Swabs
[in work]

### Diagnosis in the Lungs 

#### XRay-Imaging
Conventional XRay or computer tomography (essentially multiple XRay pictures enabling computer assisted construction of 3d image[to check])
Round glassy occupancies in the lung are characteristic for Covid [to check and cite]
(+) specific
(+) detects infection of the lung regardless whether there are viruses in the upper respiratory tract
(-) radiation exposure for patient
(-) only in diseases involving the lung significantly and thus sensitivity low
=> recommended for symptomatic risk patients of very high age (radiation exposure less relevant) with negative test in the upper respiratory tract.

#### Sputum
Overall sputum has a high detection rate: 
* over 80% in the first week of symptoms and over 70% in the second [Yang et al](#summary-yang). 
* [Wang et al](#summary-wang) observed a detection over 70%. 

#### Endotracheal Aspirate

#### BALF 
Broncho-alveolar-lavage fluid (BALF). Even though the lower respiratory tract is the most relevant location for pathogenesis, the viral load is less known especially for mild cases since sampling is complicated.
[Yang et al](#summary-yang) detected a viral load in all severe cases. The viral 


### Challenges regarding Viral Load and Diagnosis
Often investigations for viral load are biased. Reasons are:
* Sampling is biased towards symptomatic patients. Not necessarily bad since symptomatic patients are more infectious ([Section Infectors and Symptoms](./spread_analyses.md#infectors-and-symptoms) in the Chapter Spread Analyses) and more at risk for severe disease.
* Often NPS samples are taking as a 'Gold Standard' to diagnose Covid-19. Measuring viral load e.g. in saliva is often compared to NPS viral load and sometimes NPS sampling is assigned a sensitivity of 100%. 
* E.g. not to every infection in the upper respiratory tract the body responds with antibodies or only antibodies of a specific type. On the other hand a lower respiratory tract infection may yield antibodies, but be negative at a single PCR in the upper respiratory tract.



## Saliva Diagnosis
Key features of saliva based Covid diagnosis are:

* Viral Load: 
  * The viral load in saliva depends on the exact method and location of sampling (e.g. throat wash, coughing up sputum) and the time taken (e.g. the saliva flow is increased during eating and decreased at night). Overall the viral load is similar to the viral load in the nose with an agreement of about 90%. The results across studies and the collections methods they use ([Sampling Methods](#viral-load-for-different-saliva-sampling-methods)). 
  * The optimal [collection time](#saliva-collection-timing) is upon waking and before eating. In this case the viral load and the detection rate is about 10% higher than in NPS specimens ([Upon Waking Saliva](#upon-waking-saliva))
  * The viral load decrease in saliva is a predictor for the pathogenesis [Silva et al](#summary-silva).
* Suitability: 
  * Saliva is simpler than NPS and suitable for self collection.
  * Well suited for pooled analysis [citation in work]. 
  * Routine screening (e.g. in hospitality or care homes)

### Literature on Saliva Diagnosis
Summaries of the papers are in the section [Summarized References](#summarized-references).
* saliva and other diagnosis forms for Covid-19 are concisely described in the recommended introduction of ([Matic et al](#summary-matic)
* Reviews are
  * [Lee et al](#summarized-review-lee)(has nice plots comparing the different sampling methods) 
  * [da Silva](#summarized-review-dasilva)(helpful summaries of studies).
  * [Manabe et al](#summary-manabe) investigate crevicular fluid and review the literature on saliva diagnosis. 
* Well specified and carefully carried out investigations are:
  * [Wyllie](#summary-wyllie-letter). Early morning before eat saliva. **Recommended** read with illustrative plots, detailed information available in the appendix.
  * [Rao](#summary-rao). Early morning before eat saliva.
  * [Huber](#summary-huber). Throat wash specimens. 
* An interesting read is [Yokota 2](#summary-yokota-2). In his [Opinion Paper](#summary-opinion-by-azzi) L. Azzi opts for saliva based diagnosis and summarizes key points of [Yokota et al](#summary-yokota-2).
* [Hung et al](#summary-hung) describe the SARS-CoV-2 viral load in saliva at 5 different time point during the day and observe that the viral load peaks in the morning.

### Viral Load for Different Saliva Sampling Methods
[in work]
Location of sampling (e.g. throat wash, coughing up sputum) and timing  (e.g. time of day / before after eating since the saliva flow is increased during & after eating and decreased at night). Thus different [saliva Sampling Procedures](#saliva-sampling-procedures) yield different results. 

* #### Throat saliva
  Testees (mostly symptomatic or exposed) in [Huber et al](#summary-huber) were asked to clear the throat. The viral load and detection rate in NPS were higher but about 90% of NPS positive were also positive in saliva. A tendency for increasing the saliva sensitivity for triple throat washing is observed. 
* #### Coughed up Sputum/Nasal Secretions by sniffing
  * [Procop et al](#summary-procop): 216 Covid-19 symptomatic patients had enhanced saliva (provide all produceable secretions: strong sniff, elicited cough) collected and NPS analyzed. 38 were positive both for NPS and saliva. 1 testee had only saliva tested positive. 
* #### Upon Waking Saliva
  * [Wyllie et al](#summary-wyllie-letter) analyze saliva samples from hospitalized patients given each couple of days:
    * A viral load is detected in about 80% of the cases in the first 5 days after hospital admission of confirmed Covid-19 cases.
    * Both the viral load and the detection rate observed are higher than for NPS in day 0 to 5 and day 5 to 10 days after hospital admission. From day 11 and onwards about 50% are positive in NPS
  * [Rao et al](#summary-rao) observed much higher detection rates from early morning saliva than from NPS specimens in male recruits asymptomatic at the time of testing (average age 27) on day 8-10 in isolation after an initial positive NPS test. 
* #### Saline Gargle
  [Berenger et al](#summary-berenger) hast testees in a first round gargle saline water and in a second round to accumulate and then spit saliva and add universal transport medium (UTM). The sensitivity observed in the UTM method is higher than in the saline gargle method.
* #### Crevicular Fluid
  * [Manabe et al](#summary-manabe) investigated the effectiveness of crevicular fluid for Covid-19 diagnosis. 

### Notes on Saliva Diagnosis
* Saliva tests can be cheated by increasing the saliva flow. Saliva is essentially ultra-filtrated blood enriched with some enzymes and electrolytes and is secreted from cells in the different salivary glands.  

  > One reason more to relax the often uselessly strict quarantine and isolation rules across the world. Over 90% of the infected are undetected. Important is, to avoid crowded indoor spaces when infected and ventilate the flat well (most people live in flats). 

## Summarized References
As usually for summarized references:
* in "..." are word-for-word extracts. 
* "Figure 1" and similar refer to figures in the corresponding paper. 
* While trying to capture the key findings of the investigations, the summaries are biased towards the findings referred on this page. 


### Viral Load in Sputum

#### Summary Yu
**SARS-CoV-2 viral load in sputum correlates with risk of COVID-19 progression**
##### Methods
* Total 92 patients
  * 30 were severe on admission. 
  * 62 mild-moderate cases at admission, 11 cases became severe during hospitalization.
* "The sputum specimens were collected from the lower respiratory tract of each patient at admission and the levels of viral nuclei acid were determined by a real-time PCR (RT- PCR) approach and indicated by the cycle threshold (Ct) values of RT-PCR assays [2]."
##### Results
"We found that the viral load of the sputum specimen in the lower respiratory tract tested at baseline is closely related to the severity of COVID-19. More importantly, patients with a higher baseline viral load are more likely to become severe." //shown in Figure 1 of the paper. 



### Viral Load in Saliva Summarized Refs

#### Summarized Review Lee
Lee, Rose & Herigon, Joshua & Benedetti, Andrea & Pollock, Nira & Denkinger, Claudia. (2020). **Performance of Saliva, Oropharyngeal Swabs, and Nasal Swabs for SARS-CoV-2 Molecular Detection: A Systematic Review and Meta-analysis.** 10.1101/2020.11.12.20230748. 
A review and meta analysis. Helpful plots showing uniformly the detection rates of SARS-CoV-2 in NPS/saliva/NS for the reviewed studies and aggregated detection rates for different features (e.g. symptomatic) and different collection methods (NPS/saliva/NS). 
##### Methods
"We systematically searched PubMed, Google Scholar, medRxiv, and bioRxiv (last retrieval October 1st, 2020) for comparative studies of alternative specimen types [saliva, oropharyngeal (OP), and nasal (NS) swabs] versus NP swabs for SARS-CoV-2 diagnosis using nucleic acid amplification testing (NAAT)." => "From 1,253 unique citations, we identified 25 saliva, 11 NS, 6 OP, and 4 OP/NS studies meeting 15 inclusion criteria."
##### Findings
The results from the different studies/data aggregated by feature are summarized to a figure which shows % positive alternate specimens, % positive NP specimens and (if available) % specimens where both NP and alternate are positive. 
Selected aggregated results, the complete list is found in Figure 2 in the paper

> Note: Sensitivity values should be taken as a score and *not* as detection rates in percentages since diagnosis by upper respiratory specimens alone is not possible.

* Overall saliva they conclude a detection rate of 88% for saliva and 94% for NPS //both detection rates are likely much lower see Limitations
* Cough or deep  throat saliva have a detection rate of 94% while NPS have 89%.
* Asymptomatic patients have a detection rate of 87% in saliva and 73% in NPS
* Symptomatic Patients have a detection rate of 88% in saliva and 96% in NPS

 

#### Summarized Review daSilva 
Medeiros da Silva, R. C., Nogueira Marinho, L. C., de Araújo Silva, D. N., Costa de Lima, K., Pirih, F. Q., & Luz de Aquino Martins, A. R. (2020). **Saliva as a possible tool for the SARS-CoV-2 detection: A review.** Travel medicine and infectious disease, 38, 101920. 

Helpful summaries of 39 studies, which analyze the viral load in saliva. 
##### Methods
Search for (“saliva”) and (“SARS-CoV-2” or “coronavirus” or “COVID-1”) in PubMed, Medline, Cochrane Library, Web of Science, Embase and Scopus yielded:
"A total of 363 studies were identified by  and 39 were selected for review."
##### Findings
* For most studies: Saliva samples yielded a viral load and positivity rate similar to NPS samples. 
* For most studies: If NPS samples were positive, saliva samples were positive in 70% or more cases. 
* "Salivary samples for SARS-CoV-2 detection was as consistent and sensitive as the nasopharyngeal swabs in most studies, having been effective in detecting asymptomatic infections previously tested negative in nasopharyngeal samples."

#### Summarized Review Mazumder
**Diagnostic tests for SARS-CoV-2: current status and issues**
Different diagnostic tests for Covid-19 are reviewed and pro & contras discussed:
* Virus culture:
  Not recommended for routine diagnosis since: Requires skilled workers, high biosafety & 3-6 days for SARS-CoV-2 to cause apparent cytopathic effects.
* Molecular assays (references for each specimen in the paper, the investigation from [Wang et al](#summary-wang) is one of those):
  Specimen              | Sensitivity (%)
  --|--
  Nasopharyngeal swab   | 63 – 89
  Oropharyngeal swab    | 32 – 84
  Bronchoalveolar lavage| 93
  Sputum                | 72
  Blood                 | 3 – 15
  Feces                 | 10
  Anal swab             | 10-21.2
* Immunoassays (antigen tests):
  A challenge is especially specificity since: "Within a genus, antibodies against other coronaviruses might cross-react and such cross-reactive conserved viral proteins limit the use of whole virus–based assays, for example, immunofluorescence assay (IFA) [7]."
* Symptom-based diagnosis

#### Summary Manabe
**Self-collected oral fluid saliva is insensitive compared to nasal-oropharyngeal swabs in the detection of SARS-CoV-2 in outpatients**
##### Methods
* Self-collected Swabs: "Participants self-collected mid-turbinate nasal and oropharyngeal (nasal-OP) swabs; both swabs were placed in 3ml viral transport medium (VTM, 11).
* Saliva Collection: 
  * "Participants were instructed to open and remove the saliva collection sponge from the device container, rub their gums for 1-2 minutes with the sponge, then reinsert the swab back into the device container and closes the container. This collection method targets gingival crevicular fluid, which leaks from the space between the gums and teeth and is enriched with IgG antibodies derived from blood."
  * "Midway through enrolment, participants were asked to add spit into the Oracol collection tube to evaluate if that might enhance sensitivity."
* "VeroE6-TMPRSS2 [16] cell culture model was used to assess viable virus when incubated with VTM (nasal-OP samples only). SARS-CoV-2 specific growth was verified by indirect immunofluorescence for SARS-CoV-2 antigen (nucleocapsid and spike proteins).[17]"
##### Results
* In the first 5 days after symptom onset the agreement for positive samples was about 90%. But in days 5 to 10 about 60% of crevicular fluid samples where positive of the swab positive ones (as shown in Figure 3). 
* "We cultured all rRT-PCR positive nasal-OP specimens on VeroE6 TMPRSS2 cells. All culture positive samples (n=16) were obtained within 11 days of symptom onset. In all matched samples in which SARS-CoV-2 was culture positive (n=9), SARS-CoV-2 RNA was detected by rRT-PCR in both nasal-OP and oral fluid samples."
##### Discussion
* "In our study, we found that spit added to the oral fluid did increase sensitivity. Taken together spit/drool sample type is better than oral fluid from the gumline. These differences are important since many in vitro devices that are currently being tested for the direct detection of SARS-CoV-2 use different salivary sample types including passive drool, spit, oral fluid collected with a sponge, and sputum from clearing the throat."

#### Summary Azzi
Azzi, L., Carcano, G., Gianfagna, F., Grossi, P., Gasperina, D. D., Genoni, A., Fasano, M., Sessa, F., Tettamanti, L., Carinci, F., Maurino, V., Rossi, A., Tagliabue, A., & Baj, A. (2020). **Saliva is a reliable tool to detect SARS-CoV-2. The Journal of infection,** 81(1), e45–e50. https://doi.org/10.1016/j.jinf.2020.04.005
##### Methods
Saliva samples of 25 patients with severe COVID-19 are analyzed by rRT-PCR.
##### Results
* "SARS-CoV-2 was detected in all 25 patients first salivary swab, with different Ct values (range 18.12–32.23, mean value 27.16 +/− 3.07), but all of them were under the Ct value of 33."
* "Interestingly, there was an inverse correlation between the LDH values recorded in the haematochemical analyses and the Ct values, thus the viral load detected in the saliva was correlated to the tissue damage reported by biomarkers (p = 0.04) (Table 2) (Fig. 1a and b)."
* "In contrast, there was not a significant correlation between usRCT and the Ct values (p = 0.07), but an inverse tendency between this inflammatory index and the viral load detected in saliva (Fig. 1c and d) was observed."
* "A striking feature was highlighted in two patients who showed positive salivary results on the same days when their pharyngeal or bronchoalveolar swabs proved to be negative."



#### Summary Opinion by Azzi
**SALIVA IS THE KEY ELEMENT FOR SARS-CoV-2 MASS SCREENING**
Lorenzo Azzi (author of the [paper above](#summary-azzi)) gives arguments for saliva based diagnosis and possible applications such as mass screening. Key points of the [paper by Yokota et al](#summary-yokota-2) are summarized.

#### Summary Berenger
Berenger BM et al., 
**Saliva collected in universal transport media is an effective, simple and high-volume amenable method to detect SARS-CoV-2**, 
Clinical Microbiology and Infection, https://doi.org/10.1016/j.cmi.2020.10.035
##### Methods
* "PCR-positive individuals admitted to the hospital or in the community diagnosed by a NP or OP swab gave their consent"
* Collection Methods:
  1. __saliva Saline gargle__:
     * "For the saline gargle and spit method, participants took 3 mL of 0.85% sterile saline into their mouth, moved it back and forth to each side of the mouth twice, then gargled for 3 s and spat into a sterile polypropylene urine collection container (~4 cm wide)."
     * "Twenty-nine participants (58.6% hospitalized) had a saline gargle sample collected."
  2. __saliva UTM method__:
     * "[Pasomsub et al.](#summary-pasomsub) report on a low number of patients (n = 21 positives), finding a sensitivity of 84.2% for saliva collected in UTM compared with NP and oropharyngeal/throat (OP) swabs for the detection of SARS-CoV-2. We therefore decided to test the method of Pasomsub et al. in a larger number of patients." 
     * "For the UTM (universal transport medium) method, participants accumulated saliva for 1 - 2 min (no specific volume) and spat into an empty urine container, and then 3 mL of Copan Italia UTM-RT (Code 330C) was immediately added."
  3. __NP Swap__:
  "A NP swab (Flexible Mini-tip FloqSwab in 3 mL of UTM, Copan) was concomitantly collected from all patients by an infectious disease physician or a paramedic trained in NP swab collections."

##### Results
###### Saline Gargle Method
"Twenty-nine participants (58.6% hospitalized) had a saline gargle sample collected. Using a reference standard of either sample positive (n = 24), the positive agreement (95% confidence interval (CI)) for the saline gargle was 70.8% (50.8-85.1) and NP swab 95.8% (79.8-99.8). In one case, only saliva was positive. The mean time from symptom onset and study collection was 11.6 days (range 3-44)."  
saliva\Swab     | Swab + | Swab - | Total
--|--|--|--
Saliva-Gargle + |     16 |     1  |  17 
saliva-Gargle - |      7 |     5  |  12
Total           |     23 |     6  |  29 

###### saliva-UTM Method
"Seventy-five patients (9.5% hospitalized) had saliva in UTM collected. The positive agreement for saliva was 84.1% (73.7-90.9) and NP swab 91.3% (82.3-95.9)"
saliva\Swab  |   Swab + | Swab - | Total
--|--|--|--
Saliva-UTM + |       52 |    6   |  58 
saliva-UTM - |       11 |    6   |  17
Total        |       63 |   12   |  75 


#### Summary Iwasaki
Iwasaki, S., Fujisawa, S., Nakakubo, S., Kamada, K., Yamashita, Y., Fukumoto, T., Sato, K., Oguri, S., Taki, K., Senjo, H., Sugita, J., Hayasaka, K., Konno, S., Nishida, M., & Teshima, T. (2020). **Comparison of SARS-CoV-2 detection in nasopharyngeal swab and saliva.** The Journal of infection, 81(2), e145–e147. https://doi.org/10.1016/j.jinf.2020.05.071
##### Methods
Analyzed the viral nasopharyngeal and saliva samples in 76 patients.
##### Results
Detection Rate:
saliva\Swab | Swab + | Swab - | Total
--|--|--|--
Saliva +    |    8 |      1   |  9 
saliva -    |    1 |     66   | 67
Total       |    9 |     67   | 76 

Notes: 
* 8 patients were positive for both saliva and nasopharyngeal probes. 1 patient for saliva only and 1 for nasopharyngeal only. 
* A weak tendency for saliva samples to have a higher viral load early and nasopharyngeal swabs in the later phase of Covid-19 was observed. 



#### Summary Güclü
Güçlü, E., Koroglu, M., Yürümez, Y., Toptan, H., Kose, E., Güneysu, F., & Karabay, O. (2020). 
**Comparison of saliva and oro-nasopharyngeal swab sample in the molecular diagnosis of COVID-19.** Revista da Associacao Medica Brasileira (1992), 66(8), 1116–1121. https://doi.org/10.1590/1806-9282.66.8.1116
##### Methods 
Three groups of patients are analyzed:
* "Group 1 (30 patients): Hospitalized patients with a finding consistent with COVID-19 in the CT scan of the lung and detected SARS-CoV-2 by PCR in at least one ONS sample."
* "Group 2 (15 patients): Hospitalized patients with a finding compatible with COVID-19 in lung CT examination, but in whom SARS-CoV-2 were not detected in at least two ONS samples by PCR."
* "Group 3 (19 patients): Patients who present to the emergency department with complaints compatible with COVID-19 (fever, cough, shortness of breath) but have normal CT."
##### Results
* Group 1: Saliva samples were positive in 21 (70%) of the 30 patients in Group 1. All ONS samples by group construction. 
* Group 2: Two saliva samples were positive for SARS-CoV-2. No positive ONS samples by group construction.
* Group 3: SARS-CoV-2 was detected in the saliva of two patients, and only in ONS of one patient. 

Results for testees in Group 1 and Group 2 (all patients in group 1 & 2 have Covid with a very high probability):
saliva\Swab | Swab + | Swab - | Total
--|--|--|--
Saliva +    |    21 |    2    | 23 
saliva -    |     9 |   13    | 22
Total       |    30 |   15    | 45 


#### Summary Huber
**Large parallel screen of saliva and nasopharyngeal swabs in a test center setting proofs utility of saliva as alternate specimen for SARS-CoV-2 detection by RT-PCR**

##### Methods
* "Adults and children that qualified for a regular SARS-CoV-2 test according to the FOPH (Swiss federal office of public health) and reported to one of the participating test centers or emergency units were enrolled from October 20, 2020 to November 4, 2020. In total 1187 individuals (male 54.8%/female 45.2%) were included (Table1)." 
* "Participants were either asked to clear the throat thoroughly (“Basic”, N = 835) or in an intensified protocol to clear it three times (“Enhanced”, N = 352) and collect about 0.5 – 1 ml of saliva."
* "In the present study, neither eating, drinking nor smoking was controlled as study subjects came for an elective analysis by NPS and thus could only be informed about the saliva sampling on site immediately before the collection."
*  "Immediately after saliva collection, VTM (virus transport medium) was added to the crude saliva and the content mixed through gentle twisting." 
*  "Saliva was collected directly after NPS and both specimens immediately sent for SARS-CoV-2 RT-PCR testing."
##### Results
saliva\NPS  | NPS +  | NPS -  | Total
--|--|--|--
Saliva +    |    228 |    4   |  232 
saliva -    |     20 |  935   |  955
Total       |    248 |  939   | 1187 


* In Figure 4 A the measurements points are plotted in a NPS-viral-load to saliva-viral-load coordinate system. Findings:
  * People experiencing strong symptoms tend to have a higher viral load (lower CT values) both in NPS and saliva. 
  * The NPS viral load is higher for the majority of testees.
  * The viral load in saliva and NPS differs a lot for some testees.
* Figure 4 B shows that earlier after symptom onset (only symptomatic patients) patients get tested, the higher the viral load is. 


#### Summary Matic
**Practical challenges to the clinical implementation of saliva for SARS-CoV-2 detection**
The introduction section of their paper is highly recommended: it provides a concise overview of the different diagnostic methods and associated challenges.

Experiments:
1. Comparison of saliva and NPS for SARS-CoV-2 detection (Experiment 1)
2. Whether delayed analysis of saliva samples reduces detection rates (Experiment 2) 

##### Methods
__Experiment 1 Comparing NPS and saliva Diagnosis:__
* Testees: "From March–May 2020, paired saliva and nasopharyngeal swabs were received from patients under investigation for COVID-19 from various clinical settings: inpatients, residents of long-term care (LTC) facilities, healthcare workers, and outpatients."
* NPS Collection: "Nasopharyngeal swabs were collected using COPAN Flocked Swab with 2 mL of Universal Transport MediumTM (COPAN, Brescia, Italy) or 3 mL of BDTM Universal Viral Transport System (Becton Dickinson, Sparks, MD), depending on availability."
* saliva Collection: "After the nasopharyngeal swab was collected, patients were asked to provide approximately 1 mL of saliva by pooling saliva in the mouth and spitting into a sterile screw-top container (Starplex Scientific Inc., Etobicoke, Canada)."
  
__Experiment 2 Stability of SARS-CoV-2 RNA for Detection__
Five saliva samples (2 from patients and 3 from volunteers spiked with known positive NPS samples) were stored at room temperature and analyzed at delayed time points: 0, 12, 24, 36, and 48 h.

##### Results
__Experiment 1 Comparing NPS and saliva Diagnosis:__
  
saliva\Swab | Swab + | Swab - | Total
--|--|--|--
Saliva +    |     15 |     1  | 16 
saliva -    |      6 |    52  | 58
Total       |     21 |    53  | 74 

__Experiment 2 Stability of SARS-CoV-2 RNA for Detection__
The CT values of the E-gene in the saliva samples stayed constant over the time points measured. 


#### Summary Pasomsub
[in work]
**Saliva sample as a non-invasive specimen for the diagnosis of coronavirus disease 2019: a cross-sectional study**
Pasomsub E et al., Saliva sample as a non-invasive specimen for the diagnosis of coronavirus disease 2019: a cross- sectional study, Clinical Microbiology and Infection, https://doi.org/10.1016/j.cmi.2020.05.001
##### Methods
"From 27 March to 4 April 2020, we prospectively collected saliva samples and a standard nasopharyngeal and throat swab in persons seeking care at an acute respiratory infection clinic in a university hospital during the outbreak of COVID-19. Real-time polymerase chain reaction (RT-PCR) was performed, and the results of the two specimens were compared."
##### Results
saliva\NPS | NPS + | NPS - | Total
--|--|--|--
Saliva +   |    16 |   2   |   18 
saliva -   |     3 | 179   |  182
Total      |    19 | 181   |  200 


#### Summary Procop
Procop GW, Shrestha NK, Vogel S, Van Sickle K, Harrington S, Rhoads DD, Rubin BP, Terpeluk P. 2020. 
**A direct comparison of enhanced saliva to nasopharyngeal swab for the detection of SARS-CoV-2 in symptomatic patients.** 
J Clin Microbiol 58:e01946-20. [https:// doi.org/10.1128/JCM.01946-20]()
##### Methods
* Location: Cleveland Clinic, Cleveland, Ohio, USA
* Participants: "Patients with symptoms consistent with COVID-19, as determined by a licensed health care provider and reviewed by a second health care provider, were directed to a specimen collection location."
* Specimen Collection:"If the patient agreed to submit a saliva specimen for comparative purposes, they were provided with a specimen collection cup and given instructions (see the supplemental material) on how to provide the specimen while they waited in line in their car for NPS collection. The patient was instructed to “snuff” (i.e., sniff strongly) to gather any nasal secretion/mucus into the oropharynx, to cough to produce any phlegm, and then to submit these secretions and additional saliva until the specimen reached the premarked fill line on the sterile specimen collection container (also called a urine cup). We requested 3 ml of saliva/naso-oropharyngeal secretions (referred to here as “enhanced saliva”) for this study but accepted whatever volume the patient could provide."

##### Results
Detection Rate: 
saliva\Swab     | Swab + | Swab - | Total
--|--|--|--
Saliva +        | 38     | 1      | 39 
saliva -        |  0     | 177    | 177
Total           | 38     | 178    | 216 

Viral Load:
Specimen\Gene | N-Gene | RNaseP 
--|--|--|--
Saliva        | 24.2   | 23   
NPS           | 20.6   | 25.1

Notes:
* 216 of the 224 patients had analyzed both samples without errors (9 errors in total).
* Viral Loads are shown in Figure 1: "Seventy-one percent of the saliva specimens had higher CT values (i.e., lower viral loads) than the matched NPS specimens (Fig. 1B). There was little correlation between the CT values (i.e., viral loads) of the positive saliva and NPS specimens (Pearson correlation coefficient, r = 0.162; P = 0.333) (Fig. 1C)."
* Control Sample Validity: "The mean (SD) CT value of the RNase P amplification control of the NPS specimens was 25.1 (2.2) cycles, whereas the mean (SD) CT value of this target for the saliva specimens was 23.0 (1.8). The mean difference in CT values was 2.1 (95% confidence interval [CI], 1.8 to 2.4; P = 0.001), which suggests that the saliva specimens had a greater cellularity than the NPS specimens."
  

#### Summary Rao
Rao M, Rashid FA, Sabri FSAH, et al. 
**Comparing nasopharyngeal swab and early morning saliva for the identification of SARS-CoV-2.** 
Clinical Infectious Diseases : an Official Publication of the Infectious Diseases Society of America. 2020 Aug. 
DOI: [https://doi.org/10.1093/cid/ciaa1156]()
##### Methods
* Participants:
  "This prospective single center diagnostic study was conducted among 217 individuals who were tested positive for SARS- CoV-2 via NPS at a COVID-19 quarantine center, MAEPS. These selected individuals were on days 8–10 of isolation during the sampling."..."Individuals with respiratory aid were excluded."
  "All participants were male recruits and asymptomatic at the time of sampling. The median age of the participant was 27 (IQR: 18–36) years." 
* saliva Collection:
  "Before collecting swabs, individuals were asked to provide self-collected deep throat saliva sample in a sterile collection container. Instruction on self-collected deep throat saliva was announced to participants a day prior. Briefly, upon waking up, the individuals were instructed to avoid food, water, and brushing of teeth before the collection of 2 mL of saliva."
* Swab Collection:
  "As a standard protocol, NPS from individuals were collected using sterile flocked swab and placed in sterile tube containing viral transport medium (VTM)."
* Sample Processing
  * "All samples were stored at room temperature and transported to research lab at Institute for Medical Research, Kuala Lumpur, within 5 hours of sample collection for further processing."
  * In brief: Samples were inactivated at 65 C for 1 hour, nucleic acid extraction by MagNA Pure 96 system and Viral NA Small Volume extraction kit (Roche Diagnostic GmBH, Germany), from 200 μL of viral transport medium containing the NPS or 200 μL of saliva. Testing with 1-step RT-PCR of Real-Q 2019 nCoV detection kit (Biosewoom, Inc, South Korea), which detects the E-gene and RdRp of SARS-CoV-2 and human RNase gene as an internal control. 
  * "Samples were classified as positive for SARS-CoV-2 when both E-gene and RdRp primer-probe sets were detected at cycle threshold (Ct) value of <38."

##### Results
Dectection Rate: 
saliva\Swab     |  NPS + |  NPS - | Total
--|--|--|--
Saliva +        |  73    |    76  |  149
saliva -        |  11    |    57  |   68
Total           |  84    |   133  |  217

Viral Load:
Specimen\Gene | E-Gene | RdRp | RNaseP
--|--|--|--
saliva        | 30.5   | 31   | 27
NPS           | 33     | 33.5 | 29

* Figure 1 shows the Ct values of SARS-CoV-2 genes E and RdRp and the human RNaseP gene for both saliva and NPS (values shown in the table above). tto 
* Figure 2 shows that the saliva viral loads are more broadly distributed compared to the NPS viral load. The distribution of the RNaseP CT values is quite narrowly both for saliva and NPS, indicating a uniform sampling process for both specimens.


#### Summary Wang
**Detection of SARS-CoV-2 in Different Types of Clinical Specimens**
##### Methods
* "We investigated the biodistribution of SARS-CoV-2 among different tissues of inpatients with coronavirus disease 2019 (COVID-19) diagnosed based on symptoms and radiology and confirmed by SARS-CoV-2 detection."
* "Most of the patients presented with fever, dry cough, and fatigue; 19% of patients had severe illness." 
* "A cycle threshold value less than 40 is interpreted as positive for SARS-CoV-2 RNA." 
* "Four SARS-CoV-2 positive fecal specimens with high copy numbers were cultured, and then electron microscopy was performed to detect live virus."
##### Results
* Positive rates in decreasing order:
  * Bronchoalveolar lavage fluid (14 of 15; 93%)
  * Sputum (72 of 104; 69%)
  * Nasal swabs (5 of 8; 62.5%)
  * Fibrobronchoscope brush biopsy (6 of 13; 46%)
  * Pharyngeal swabs (126 of 398; 32%)
  * Feces (44 of 153; 29%)
  * Blood (3 of 307; 1%)
  * Urine specimens (0 of 72; 0%) 
* Two of the four cultured fecel samples showed signs of virus replication.


#### Summary Williams
**Saliva as a non-invasive specimen for detection of SARS-CoV-2**
##### Methods
* Between 25th March and 1st April 2020 in Melbourne, 622 patients provided NPS samples, 522 of those 622 patients provided additionally saliva samples for Covid-19 tests. 
* Collection Methods:
  saliva: Patients were asked to pool saliva in their mouth for 1-2 minutes prior to collection.
##### Results
* 39 of 622 (6.3%) patients had PCR-positive NPS, and 33 of those 39 NPS positive (84.6%) had SARS-CoV-2 detected in saliva.
* 1 positive of 50 (2%) saliva sample of patients whose NPS sample was negative (only the saliva of 50 patiens whose NPS sample was negative was analyzed).
##### Limitations
Data not uniform/consistent:
* Of the 622 patients with NPS samples only 522 provided saliva samples (but not written if the positive 39 belonged to the 522).
* Only 50 saliva samples of those tested negative by NPS were analyzed.  


#### Summary Vaz
**Saliva is a reliable, non-invasive specimen for SARS-CoV-2 detection**
##### Methods
155 Participants with signs/symptoms suggesting SARS-CoV-2 infection underwent a nasopharyngeal swab (NPS) and/or oropharyngeal swab (OPS) and saliva collection in Salvador, Brazil. 
##### Results
saliva\NPS or OPS| NPS/OPS +  | NPS/OPS - | Total
--|--|--|--
Saliva +         |    67      |       2   |  69 
saliva -         |     4      |      82   |  86 
Total            |    71      |      84   | 155 


#### Summary Yokota 1
**Equivalent SARS-CoV-2 viral loads between nasopharyngeal swab and saliva in symptomatic patients**
##### Methods
* "Forty-two patients diagnosed with COVID-19 by positive qRT-PCR by NPS were enrolled in this study."
* "Paired NPS and saliva samples were simultaneously collected from all patients upon hospital admission between June 12, 2020 and August 6, 2020."
* "Self-collected saliva was diluted 4-fold with phosphate buffered saline and centrifuged at 2000 × g for 5 min to remove cells and debris. RNA was extracted from 200 μL of the supernatant or nasopharyngeal swab samples using QIAsymphony DSP Virus/Pathogen kit and QIAamp Viral RNA Mini Kit (QIAGEN, Hilden, Germany)."
##### Results
saliva\Swab | NPS + | NPS - | Total
--|--|--|--
Saliva +    |    34 | 4     | 38 
saliva -    |     0 | 4     | 4
Total       |    34 | 8     | 42 


#### Summary Yokota 2
**Mass screening of asymptomatic persons for SARS-CoV-2 using saliva**
Yokota, I., Shane, P. Y., Okada, K., Unoki, Y., Yang, Y., Inao, T., Sakamaki, K., Iwasaki, S., Hayasaka, K., Sugita, J., Nishida, M., Fujisawa, S., & Teshima, T. (2020). Mass screening of asymptomatic persons for SARS-CoV-2 using saliva. Clinical infectious diseases : an official publication of the Infectious Diseases Society of America, ciaa1388. Advance online publication. https://doi.org/10.1093/cid/ciaa1388
##### Methods
Individuals from two cohorts of asymptomatic persons: the contact tracing cohort and the airport quarantine cohort had specimens from nasopharyngeal swabs (NPS) and saliva samples analyzed by PCR.
##### Results
* "In this mass-screening study including 1,924 individuals, the sensitivity of nucleic acid amplification testing with nasopharyngeal and saliva specimens were 86% (90%CI:77-93%) and 92% (90%CI:83-97%), respectively, with specificities greater than 99.9%. The true concordance probability between the nasopharyngeal and saliva tests was estimated at 0.998 (90%CI:0.996-0.999) on the estimated airport prevalence at 0.3%. In positive individuals, viral load was highly correlated between NPS and saliva."
* "Both nasopharyngeal and saliva specimens had high sensitivity and specificity. Self- collected saliva is a valuable specimen to detect SARS-CoV-2 in mass screening of asymptomatic persons."

###### Contact Tracing Cohort
saliva\Swab | NPS + | NPS - | Total
--|--|--|--
Saliva +    | 38    |   6   |  44 
saliva -    |  3    | 114   | 117
Total       | 41    | 120   | 161

###### Airport Quarantine cohort
saliva\Swab | NPS + | NPS - | Total
--|--|--|--
Saliva +    |  4    |    0  | 4 
saliva -    |  1    | 1758  | 1759
Total       |  5    | 1758  | 1763 




### References RNA Detection Stability

#### Summary Ott
**Simply saliva: stability of SARS-CoV-2 detection negates the need for expensive collection devices**

##### Methods
* "RNA was extracted from saliva samples collected from COVID-19 inpatients and healthcare workers at the Yale-New Haven Hospital(...)"
* Experiment 1: "RNA templates were tested by RT-qPCR for SARS- CoV-2 RNA (N1) on day of collection (~12 hours post collection) and at various time points after the storage of the remaining, unsupplemented samples at temperatures of -80°C, -20°C, +4°C, 19oC, or 30°C."
* Experiment 2: Cell Culture Test: Vero-E6 cells were inoculated with saliva samples.

##### Results
* Experiment 1 __Detection Statiblity__
   * Supplemental Figure 1: The Ct values of the viral RNA sequence for the N1 gene was not much different after storing at the different conditions (- 80°C freeze, +4°C, room temperature  and 30°C) for several days. 
   * "Supplemental Figure 2. Detection of human RNAse P (RP) declines over time when stored in saliva in warmer conditions."
* Experiment 2 __Infectable Virions__
   Unclear if virions replicated: For some cultures a lower Ct value was observed (indicating replication), however follow up checking by plague assays was negative.



### Time Series of Viral Load in Saliva


#### Summary Silva
[in work]
**Saliva viral load is a dynamic unifying correlate of COVID-19 severity and mortality**
##### Methods
* Continuation of the saliva analyses done in [Wyllie et al](#summary-wyllie-letter). The obtained time series of viral loads in saliva and NPS specimens are compared to the pathogeneses. 
##### Results
* The faster the saliva viral load decreases the better the prognosis (shown in Figure 2 D).
* NPS viral load peaks later than the saliva viral load and is quite variable (Figure 2 J). 

#### Summary To
**Temporal profiles of viral load in posterior oropharyngeal saliva samples and serum antibody responses during infection by SARS-CoV-2: an observational cohort study**
[in work]

#### Summary Wyllie
**Saliva is more sensitive for SARS-CoV-2 detection in COVID-19 patients than nasopharyngeal swabs**

> updated results with more participants in the [letter version](#summary-wyllie-letter) below

##### Methods
* Participants: 
  * Group In-patients: 44 COVID-19 inpatients.  29 of those 44 had paired samples taken (Table 1). Initial Covid-19 diagnosis at hospital admission by NPS for all patients. 
  * Group Healthcare Workers: 485 asymptomatic healthcare workers (e.g., without fever or respiratory symptoms) with occupational exposure to patients with COVID-19 were enrolled.
* Sample Collection: 
  * saliva: "Saliva samples were self-collected by the patient. Upon waking, patients were asked to avoid food, water and brushing of teeth until the sample was collected. Patients were asked to repeatedly spit into a sterile urine cup until roughly a third full of liquid (excluding bubbles), before securely closing it."
  * NPS: "Nasopharyngeal samples were taken by registered nurses using the BD universal viral transport (UVT) system. The flexible, mini-tip swab was passed through the patient's nostril until the posterior nasopharynx was reached, left in place for several seconds to absorb secretions then slowly removed while rotating."
* Sample Handling: "All samples were stored at room temperature and transported to the research lab at the Yale School of Public Health within 5 hours of sample collection."
  
##### Results

Detection Rate: 
saliva\NPS       | NPS +  | NPS - | Total
--|--|--|--
Saliva +         |   24   |    8  | 32
saliva -         |    3   |    3  |  6 
Total            |   27   |   11  | 38

Notes:
* "As results from N1 and N2 were comparable (​Extended Data Fig. 1​), all virus copies are shown as calculated using the N1 primer-probe set."
* In Figure 1 in their paper the viral loads from saliva and NPS are compared. The mean viral load in saliva is about 5 times higher than in NPS. In Figure 1c the paired viral load points are plotted. The plot corresponds to the Detection Rate table shown above. 
* In Figure 2 the measurements of viral titers are shown as time series per patient. The viral load is less variable i.e. the decrease of the viral load in saliva is more monotone compared to NPS. 


#### Summary Wyllie Letter
**Saliva or Nasopharyngeal Swab Specimens for Detection of SARS-CoV-2**
This is a more up to date version of the [paper above](#summary-wyllie) published as letter on nejm.org.

> This letter motivated me to recommend & read about saliva diagnosis. Key information such as sampling methods is in the appendix of the letter (which I first overlooked and thus summarized the paper version first) and even some raw data.

##### Methods
to be done (similar to the [paper version above](#summary-wyllie))
* Participants:
  * In-Patients: tested positive for SARS-CoV-2 by nasopharyngeal and/or oropharyngeal swab with CLIA-approved assay
  * "Upon waking, patients were asked to avoid food, water and brushing of teeth until the sample was collected. Patients were asked to repeatedly spit into a sterile urine cup until roughly a third full of liquid (excluding bubbles), before securely closing it. All samples were stored at room temperature and transported to the research lab at the Yale School of Public Health within 5 hours of sample collection and tested within 12 hours of sample collection."
* Detection limits for SARS-CoV-2 using the US CDC “N1” assay is at cycle threshold 38, which corresponds to 5,610 virus copies/mL 
  
##### Results

__In-patients:__
* __Figure 1 A:__
  * "Using primer sequences from the Centers for Disease Control and Prevention, we detected more SARS-CoV-2 RNA copies in the saliva specimens (mean log copies per milliliter, 5.58; 95% confidence interval [CI], 5.09 to 6.07) than in the nasopharyngeal swab specimens (mean log copies per milliliter, 4.93; 95% CI, 4.53 to 5.33) (Fig. 1A, and Fig. S1 in Supplementary Appendix 1)." //=> about 4.4 times higher in saliva
  * The viral load in saliva can be as high as 10^10 copies/milliliter.
* __Figure 1 B:__ "At 1 to 5 days after diagnosis, 81% (95% CI, 71 to 96) of the saliva samples were positive, as compared with 71% (95% CI, 67 to 94) of the nasopharyngeal swab specimens." 
* __Figure 1 C and D:__
  * "In three instances, a negative nasopharyngeal swab specimen was followed by a positive swab at the next collection of a specimen (Fig. 1D); this phenomenon occurred only once with the saliva specimens (Fig. 1C)."
  * "During the clinical course, we observed less variation in levels of SARS-CoV-2 RNA in the saliva specimens (standard deviation, 0.98 virus RNA copies per milliliter; 95% credible interval, 0.08 to 1.98) than in the nasopharyngeal swab specimens (standard deviation, 2.01 virus RNA copies per milliliter; 95% credible interval, 1.29 to 2.70) (see Supplementary Appendix 1)."
  * " In specimens collected from in-patients by health care workers, we found greater variation in human RNase P cycle threshold (Ct) values in nasopharyngeal swab specimens (standard deviation, 2.89 Ct; 95% CI, 26.53 to 27.69) than in saliva specimens (standard deviation, 2.49 Ct; 95% CI, 23.35 to 24.35)."

Detection Rate across all days: 
saliva\NPS       | NPS +  | NPS - | Total
--|--|--|--
Saliva +         |   31   |   12  | 43
saliva -         |    8   |   19  | 27 
Total            |   39   |   31  | 70 

Ct values:
Specimen\Gene | N Gene | RNaseP(approx) 
--|--|--
saliva        | 28.7   | 24   
NPS           | 31.3   | 29

__Healthcare workers:__
* "We detected SARS- CoV-2 RNA in saliva specimens obtained from 13 persons who did not report any symptoms at or before the time of sample collection. Of these 13 health care workers, 9 had collected matched nasopharyngeal swab specimens by themselves on the same day, and 7 of these specimens tested negative (Fig. S2). The diagnosis in the 13 health care workers with positive saliva specimens was later confirmed in diagnostic testing of addi- tional nasopharyngeal samples by a CLIA (Clinical Laboratory Improvement Amendments of 1988)– certified laboratory."

saliva\NPS       | NPS +  | NPS - | Total
--|--|--|--
Saliva +         |   2   |   7    | 9
saliva -         |   0   |        |  
Total            |   2   |        |  

__In-patients compared to Healthcare workers:__
* "In specimens collected from in- patients by health care workers, we found greater variation in human RNase P cycle threshold (Ct) values in nasopharyngeal swab specimens (standard deviation, 2.89 Ct; 95% CI, 26.53 to 27.69) than in saliva specimens (standard deviation, 2.49 Ct; 95% CI, 23.35 to 24.35)."



### Time Series Viral Load in Multiple Locations

#### Summary Yang
**Laboratory Diagnosis and Monitoring the Viral Shedding of SARS-CoV-2 Infection**
[https://doi.org/10.1016/j.xinn.2020.100061]()

Location: Shenzhen; Date: written in May, patients likely from January to March
##### Methods
* Participants: "Four hundred and ten patients with confirmed COVID-19 from the Guangdong CDC (Center for Disease Control and Prevention), who were hospitalized in Shenzhen Third People’s Hospital, were included."
* "3,552 respiratory specimens from 410 patients were serially collected during hospitalization, including 559 oropharyngeal swabs, 2,231 nasopharyngeal swabs, 696 sputum samples, and 66 BALF samples."
* "The different types of specimens from laboratory-confirmed COVID-19 cases were stratified into three groups based on the collection times: 0–7, 8–14, and R15 d.a.o."

##### Results

It is observed that NPS have quite a low detection rate (only about 60% for mild cases) and sputum has a high detection rate for both mild and severe cases (about 80%). Detection rates adapted and shortened from Table 1 in the paper:

###### Positive rate (n/N, %) during day 0 - 7 of illness
Specimen      | Severe Cases | Mild Cases
--|--|--
oropharyngeal | 40/55 (72.7) | 84/158 (53.2)
nasopharyngeal| 58/68 (85.3) | 195/314 (62.1)
sputum        | 14/16 (87.5) | 38/46 (82.6)
BALF          | 2/2 (100.0)  | 0/0 (0)

###### Positive rate (n/N, %) during day 8 - 14
Specimen      | Severe Cases | Mild Cases
--|--|--
oropharyngeal | 39/81 (48.1) | 48/105 (45.7)
nasopharyngeal| 117/170(68.8)| 241/454 (53.1) 
sputum        | 29/39 (74.4) | 80/109 (73.4)
BALF          | 13/13 (100)  | 0/2 (0)

###### Ct values (median, range) during day 0 - 7
Specimen      | Severe Cases | Mild Cases
--|--|--
oropharyngeal | 29.5 (18-36) | 29.5 (15-37)
nasopharyngeal| 29.3 (19-38) | 29 (15-38)
sputum        | 26 (19-32)   | 28 (18-38)
BALF          | 22.5 (21-24) | -


* Table 1 
* Figure 2 shows time series of viral loads for a series 21 individuals (18 severe and 3 mild patients). The detection rates vary for different time points, for different patients and for different locations:
  * While *no* viral RNA was detected in the upper respiratory tract, at the *same* time it was positive in BALF for the cases 01, 03, 07, 11, 12, 13, and 16 (Figure 2).  
  * "Viral RNA was tested negative in the first three or all of the upper respiratory samples. However, viral RNA was detected either by the further detection of upper respiratory samples (cases 21–28) or BALF samples (cases 02, 04, 06, and 07) (Figures 2 and S1)."
  * The dection of SARS-CoV
* Figure 3 shows CT Scans of the mentioned 11 cases (02, 04, 06, 07 and 21-28) tested negative at least 3 time in the upper respiratory tract. The CT  show typical ground-glass opacity in the lungs, suggesting a viral pneumonia.



#### Summary Zhang
**The SARS-CoV-2 RNA with mild pulmonary consolidation lasts longer in non- severe COVID-19 patients: an observational study**


### Viral Load Nose and Throat Series


#### Summary Yilmaz
**Upper Respiratory Tract Levels of Severe Acute Respiratory Syndrome Coronavirus 2 RNA and Duration of Viral RNA Shedding Do Not Differ Between Patients With Mild and Severe/Critical Coronavirus Disease 2019**
##### Methods
* "We collected serial upper respiratory tract samples (1 nasopharyngeal swab and 1 throat swab put in a single collection tube with 1 mL of transport medium) for real-time PCR of SARS-CoV-2 RNA for all patients."
* " Severe/critical patients were defined as those requiring invasive mechanical ventilation or high-flow nasal oxygen and mild as those not requiring supplementary oxygen or hospitalization."
* "Viral load, expressed as log10 of viral RNA per swab, was calculated as (47 – observed cycle threshold value) / 3.4. This formula applies the average of the parameters (slope and constant) that we observe when we quantify viruses in serum using quantitative PCR with serial dilution of quantification standards (plasmid carrying target sequence) and presumes a 10-μL sample volume in the reaction and a 97% efficiency in the PCR." 
##### Results
* "The highest viral loads were observed early after onset of symptoms in both groups of participants (Figure 1A and 1B). At 7 days, mean viral load among virus-positive patients was 5.8 log10 copies/swab for those with mild disease (n = 37) and 5.5 log10 copies/swab for those with severe/critical disease (n = 12) (P = .53). At 14 days, mean viral load was 4.4 log10 copies/swab for those with mild disease (n = 38) and 4.2 log10 copies/swab for those with severe/critical disease (n = 14) (P = .56)."
* "The median duration of viral RNA shedding was 24.0 days in patients with mild disease and 22.5 days in patients with severe/critical disease (Figure 1C)."
* "We recorded clinical symptoms for 34 of the 39 participants with mild disease. Follow-up was done by regular phone calls. The most common presenting symptom was cough and fever, both occurring in 27 of 34 participants (79%), followed by muscle pain in 22 (65%), rhinitis in 20 (59%), and sore throat in 16 (47%)."


#### Summary Zhou
**SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients**
##### Methods
"We analyzed the viral load in nasal and throat swabs obtained from the 17 symptomatic patients in relation to day of onset of any symptoms (Fig. 1C)." 
##### Results
* "Higher viral loads (inversely related to Ct value) were detected soon after symptom onset, with higher viral loads detected in the nose than in the throat." 
* "Our analysis suggests that the viral nucleic acid shedding pattern of patients infected with SARS-CoV-2 resembles that of patients with influenza4 and appears different from that seen in patients infected with SARS-CoV."
* "The viral load that was detected in the asymptomatic patient was similar to that in the symptomatic patients, which suggests the transmission potential of asymptomatic or minimally symptomatic patients." //not necessarily the lower respiratory tract is likely more relevant for spread
* For many patients the time series of their nasal and throat swabs are varying and don't have single local maxima but several. The viral load can even be undetectable one day and be positive the next day. //=> to exclude infection several negative samples are needed.



### Flow Patterns of Body Fluids

#### Summary Carlson-Jones
**The microbial abundance dynamics of the paediatric oral cavity before and after sleep**
##### Methods 
"Using flow cytometry, bacterial and virus-like particle (VLP) abundances were enumerated for 6 oral microhabitats before and after sleep in 10 healthy paediatric sleepers."
##### Results
* Both bacteria and viral like particles live in micro habitats in the mouth. The habitats are more connected before sleep than after sleep (which is expected due the reduced saliva flow at night).  Shown in Figure 8. 
* During sleep there about a 4 to 10 fold increase in *viral particles* in the mouth (shown in Table 2). E.g. 
  * at the back of the tongue from 2.2 * 10 ^ 7 to 9.2 * 10 ^ 7 
  * at the posterior buccal vestibule from 5.7 * 10^6 to 5.7 * 10^7
* During sleep there about a 3 to 6 fold increase in *bacteria* in the mouth (shown in Table 1). E.g. 
  * at the back of the tongue from 2.9 * 10 ^ 7 to 1.3 * 10 ^ 8 
* The viral like particles are more homogenously distributed throughout the habitats than the bacteria (Table 1 and Table 2):
  * VLPs: maximal a 12 fold difference before sleep, max 6 fold diff after sleep
  * Bacteria: maximal a 40 fold difference before sleep, max 30 fold diff after sleep


#### Summary Dawes
**Salivary flow patterns and the health of hard and soft oral tissues**
* "The salivary flow rate is influenced by a large number of factors, including the degree of hydration, body position, exposure to light, previous stimulation, circadian and circannual rhythms, gland size and drug use.(7)"
  * "During sleep, the salivary flow rate is negligible.(8)"
  * "The unstimulated flow rate averages 0.3 to 0.4 milliliter per minute, but the range is wide."
  * "With gum chewing, the flow rate peaks at about 6 mL/minute in the first minute."
  * "However, Watanabe and Dawes (12) found that when subjects ate several foods, the mean flow rates during chewing varied between 3.15 and 4.94 mL/minute, while infusion of 5 per- cent citric acid into the mouth elicited a flow rate of 7.07 mL/minute. Taste stimulation is a much more effective salivary stimulus than is chewing alone.(13)"
* "Because saliva from the various glands enters the mouth at several locations, it is not well-mixed. The enzyme amylase is secreted primarily by the parotid glands, and, by using amylase as a marker for parotid saliva, Sas and Dawes (5). 
* "When flow is unstimulated, the parotid, sub- mandibular, sublingual and minor mucous glands (MMGs) contribute about 25 percent, 60 percent, 7 to 8 percent and 7 to 8 percent, respectively, to whole saliva, but when flow is stimulated, the parotid glands’ contribution increases by at least 10 percent."


#### Summary Hung
**Early-Morning vs Spot Posterior Oropharyngeal Saliva for Diagnosis of SARS-CoV-2 Infection: Implication of Timing of Specimen Collection for Community-Wide Screening**
##### Methods 
"Posterior oropharyngeal saliva was collected at 5 different time points within the same day from 18 patients with pre- viously confirmed SARS-CoV-2 infection by molecular testing. Cycle threshold (Ct) values were compared."
##### Results
* "There was an overall trend of lower Ct values from specimens collected in the early morning, with a gradual decrease of viral load towards nighttime, but reaching statistical significance only when compared with the specimens collected at bedtime. Eight out of 13 subjects had a higher viral load in the early morning than the rest of the 4 time points (before lunch, before teatime at 3 pm, before dinner, before bedtime)."
* "The distribution of Ct values at 5 different time points is shown in Figure 1. The medians (IQRs) at early morning, before lunch, before teatime at 3 pm, before dinner, and before bedtime were 34.5 (32.5–41), 38.2 (33.9–41), 36.3 (34.5–41), 41 (34.7–41), and 41 (34.7–41), respectively."

## Appendix

### Calculation of Viral Load from CT_Value
`viral_load = d_c_h / 2^CT_cycles` where `d_c_h = detection_limit_for_helper_method` (the helper method is can be optical fluorescence)
Often the viral load is given as logarithm. For logarithm to base 2:
`log2(viral_load) = log2(d_c_h) - CT_cycles`
The viral load as logarithm to base 10:
`log10(viral_load) =log2(viral_load)/log2(10) = (log2(d_c_h) - CT_cycles)/3.32`
Detection limit:
`PCR_detection_limit * 2^max_cycles = helper_method_detection_limit`

An example formula for their apparatus is in [Yilmaz et al](#summary-yilmaz).