
# Hypotheses about Nidovirales
Hypotheses about Nidovirales are discussed. They extend the [page on coronaviruses](./coronavirus.md) from a subject point. In the flavour they belong to the section [Ideas to Explore](./index.md#ideas-to-explore)
## Hypotheses in Short
[in work]

## Relevance
To know the evolutionary hosts is key to understand Nidovirales. A thorough understanding in turn can enable us to steer coronaviruses infections including SARS-CoV-2.
* #### Analyse & Understand
    Knowing and be able to observe the prehuman hosts is useful for research to better understand SARS-like coronaviruses: 
    - Better understand transmission and evolution of SARS-like viruses.
    - Analyse the interactions of SARS-like in their evolutionary hosts to better understand pathogenesis and possible benefits/risks
    - Analyse the immune responses of the evolutionary hosts to develop medications. E.g. which are the most capable antibodies. 
* #### Model System
    If they are domestic or cultivatable animals e.g. goats or insects, they provide an ideal investigation system. Especially insects would have some advantages:
    - The higher the brain is developed the more care should be take to animal well-being. Insects have only a rudimentary brain, which makes it easier to justify and do research than with mammals.
    - If coronaviruses can be cultured in insects some genes required for the mammalian tropism can be removed and so decrease the risk for lab accidents. 
* #### Direct Medications 
    The prehuman host of SARS-like in their natural habitat could even provide direct access to medications. Antiviral agents (e.g. antibodies) produced by the hosts. If there antibodies or other viral agents neutralizing virions such agents could be directly used as medications. 



# Summaries and References
[in work]
The focus of the summaries is in the following order:
1. capture the findings used in the above argumentation
2. capture the essence of the studies

Notes:
* references like 'Figure 2' refer to 'Figure 2' in the corresponding paper. 
* word by word extracts are between "..." and denoted as extracts



# Summarized References
* [Hypotheses in Short](#hypotheses-in-short)
* [Relevance](#relevance)
* [Coronaviruses Jumps](#coronaviruses-jumps)
  * [summary dudas 2017](#summary-dudas-2017)
  * [summary guan 2003](#summary-guan-2003)
* [Observations in Wild Animals](#observations-in-wild-animals)
  * [summary fan 2019](#summary-fan-2019)
  * [summary zhou](#summary-zhou)
* [Features and Evolution of SARS-like](#features-and-evolution-of-sars-like)
  * [summary anderson](#summary-anderson)
  * [summary wang 2017](#summary-wang-2017)
  * [summary corman 14](#summary-corman-14)
  * [summary ye,jin](#summary-yejin)
  * [summary ge 2013](#summary-ge-2013)
  * [summary zhang](#summary-zhang)
  * [summary k xiao](#summary-k-xiao)
  * [summary t lam](#summary-t-lam)
  * [summary boni,lemey](#summary-bonilemey)
* [Species Tropism](#species-tropism)
  * [summary zhai](#summary-zhai)
  * [summary wu](#summary-wu)
* [Coronaviruses and Insects](#coronaviruses-and-insects)
  * [Summary Tengs,Jonassen](#summary-tengsjonassen)
  * [Summary Watson 2002](#summary-watson-2002)



## Coronaviruses Jumps

### summary dudas 2017
**MERS-CoV spillover at the camel-human interface** Gytis Dudas, Luiz Max Carvalho, Andrew Rambaut, Trevor Bedford; elifesciences.org
#### Methods
The phylodynamics of 274 MERS genomes, 174 from human infections and 100 from camel infections is constructed by simulation using structured coalescent models which treat camels and humans populations as different virus habitats. 
#### Results
* camels are the reservoir hosts of MERS-CoV: 
    * the evolution of MERS takes place in camels and not in humans. 
    * human are terminal hosts and the reproduction/spread value is about 0.7 
    * the sequence data alone indicate at least 50 zoonotic introduction of MERS to humans. Extrapolation to all labraratory confirmed MERS cases yields hundreds of introductions to humans.
* frequent recombination events occur in the camels. 
* most transmissions to humans occur in the summer months, however large clusters of secondary cases from human to human transmission tend to occur in winter or spring. 

### summary guan 2003
**Isolation and Characterization of Viruses Related to the SARS Coronavirus from Animals in Southern China**
#### Methods
Investigation of several animals in a live-animal retail market in Shenzhen. 
* the animals originated from different regions of southern China and had been kept in separate storehouses before arrival to the market.  
* animals tested: 3 beaver (Castor fiber); 2 Chinese ferret-badger (Melogale moschata); 4 Chinese hare (Lepus sinensis); 1 Chinese muntjac (Muntiacus reevesi); 4 domestic cat (Felis catus); 3 hogbadger (Arctonyx collaris); 6 Himalayan palm civet (P. larvata); 1 raccoon dog (N. procyonoides)
#### Results
All of the racoon dogs and the palm civets showed signs of recent SARS infections. One of the ferrets badgers had antibodies. None of the other animals showed clear signs of a recent SARS infection.



## Observations in Wild Animals

### summary fan 2019
**Bat Coronaviruses in China** Y Fan, Z-L Shi, P Zhou et al, WIV 2019
Review bat coronaviruses in China with a focus on: viral diversity, reservoir hosts, geographical distributions and cross-species transmission potential.
* SARSr-CoVs are present in different bat species of the two families Rhinolophidae (horseshoe) and Hipposideridae.
* SARSr-CoVs that are capable of using human ACE2 were found were found in R sinicus in the Yunnan Province. Other SARSr-CoVs that cannot use human ACE2 are more widespread and distributed across multiple provinces:  Jilin, Shaanxi, Hubei, Zhejiang, Yunnan, Guizhou and Guangdong
* SARSr-CoV bat viruses are very divers in the sense that some have only sequence similarities of about 70% (Figure 2)
* "Rhinolophus bats interact constantly  with both livestock and humans in China, also harbor SARSr-CoVs"

### summary zhou
**A Novel Bat Coronavirus Closely Related to SARS-CoV-2 Contains Natural Insertions at the S1/S2 Cleavage Site of the Spike Protein**; Hong Zhou, Xing Chen, Tao Hu, Juan Li, Hao Song, Yanran Liu, Peihan Wang, Di Liu, Jing Yang, Edward C. Holmes, Alice C. Hughes, Yuhai Bi, and Weifeng Shi
#### Methods
* Collection of 302 samples from 227 different bats in two caves in Mengla County, Yunnan, between May and October 2019 (Cave a: Xishuangbanna (101.27E, 21.91N), Cave b:[coming]) 
* Next-generation metagenomic sequencing and TaqMan-based qPCR on the results
* analysis of the sequences obtained to SARS-CoV-2 and SARS-2-like using Simplot, RAxML and other tools. 
#### Results
* discovery of two novel bat coronaviruses named: RmYN01 and RmYN02
* Table 1 and Figure 1 compare the sequence of SARS-CoV-2 to SARS-2-close viruses:
    - RmYN02 is the virus most close to SARS-CoV-2 in most parts of the genome (Figure 1 A) but not in the spike gene (Figure 1 B). 
    - RaTG13 has nearly as many closest parts as RmYN02. Since RaTG13 is closer in the spike protein it is overall the genetically closest discovered virus (as of 31.8.20) to SARS-CoV-2 with an overall sequence identity of 96.1%.
    - in the RBD of the spike the closest relative is a pangolin/GD/2019
* Phylogenetic tree of SARS-2-close constructed with the whole genome, the spike, the RBD, RdRp. Since the distance vary through genome regions, the trees vary in structure. The varying distances/tree structures indicate recombinations.



## Features and Evolution of SARS-like

### summary anderson 
**The proximal origin of SARS-CoV-2**, Kristian G. Andersen, Andrew Rambaut, W. Ian Lipkin, Edward C. Holmes and Robert F. Garry; Nature Medicine

Review of distinguishing features of Sars-Cov-2 and theories about the origin:
* Notable features of the SARS-CoV-2 genome
    1. Mutations in the receptor-binding domain of SARS-CoV-2
    2. Polybasic furin cleavage site flanked by O-linked glycans: The polybasic cleavage site with the three adjacent O-linked glycans are a distinguishing feature of SARS-CoV-2 in the lineage B betacoronaviruses, however lineage A viruses have it too.
* Theories of SARS-CoV-2 origins
    1. Natural selection in an animal host. 
        * Similar RBD found in related viruses in pangolin indicate animals played a role in the adaption to human ACE2. 
        * Coronaviruses in bats and other species are  undersampled.
        * High population density and ACE2 similar to human are likely necessary in the hosts where adaption to humans has taken place. 
    2. Natural selection in humans following zoonotic transfer: A sufficiently large cluster of cases is needed to trigger the surveillance system
    3. Selection during passage: This option is considered unlikely mainly for evolutionary reasons.





### summary wang 2017
**Serological Evidence of Bat SARS-Related Coronavirus Infection in Humans, China** - N Wang, Zheng-Li Shi et al, WIV 2017
#### Methods 
Serum samples of 218 residents from rural villages in the Yunnan province were were analyzed. The villages are close to the Yanzi and Shitou caves which are inhabited by bats infected with Sars-Cov-like viruses infected bats have been found. The nucleocapsid protein of the bat coronavirus Sars-CoV-Rp3 was used to develop a SARSr-CoV specific ELISA for serosurveillance. 
#### Results: 
The sera of 6 person between 45 and 57 years old were seropositive - corresponding to a positive rate of 2.7%.

### summary corman 14
**Characterization of a Novel Betacoronavirus Related to Middle East Respiratory Syndrome Coronavirus in European Hedgehogs**
#### Methods
* analyzing of 248 fecal samples from wild European hedgehogs for CoVs
* investigate tissue probes of 27 hedgehogs that died in animals shelters
#### Results

* ##### Fecal samples: 
    146 of the 248 fecal samples (59%) were positive for a MERS-like coronavirus, named EriCoV
* ##### Tissue samples:
    12 of the 27 dead hedgehogs had viral loaded tissue probes: 
    - all 12 had a high viral load in the gut
    - RNA in other tissues was 10 fold lower but still at about 10^5/ml if observed, but not all animals had viruses in all tissues
    - 6 of the 12 MERS RNA positive and 7 of the 14 MERS RNA negative hedgehogs had MERS antibodies (by staining of MERS-CoV infected Vero cells) 
        > Somewhat intriguing that about 50% of RNA positive animals had no detectable antibodies and moreover antibody positivity seems to be independent of RNA positivity. 

### summary ye,jin 
**Zoonotic origins of human coronaviruses** - Zi-Wei Ye, Dong-Yan Jin et al, Hong Kong 2020
#### Review
The seven human coronaviruses are reviewed and compared with a focus on:
* their discovery, the zoonotic origins and interspecies transmissions.
* clinical features of infections and transmission routes
* evolution including recombination events
#### Extracts
- "many patients with MERS develop acute renal failure"
- "civets from the wild or farms without exposure to the live animal markets were largely negative for SARS-CoV"
- "Live MERS-CoV identical to the virus found in humans was isolated from the nasal swabs of dromedary camels, further indicating that camels serve as the bona fide reservoir host of MERS-CoV [56]. It is also noteworthy that generally mild symptoms but massive virus shedding were observed in camels experimentally infected with MERS-CoV [57]. Notably, infected camels shed viruses not only through respiratory route but also through fecal-oral"



### summary ge 2013
**Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor.** Ge, Shi et al - WIV - Nature 2013
#### Methods
Bats in the two caves Yanzi and Shitou in northern Yunnan were analyzed for coronaviruses infections.
#### Results
* Diverse bat SARSr-CoVs and evidence for coinfection with two or more SARSr-CoVs in the same bat are found. 
* SARSr-CoV strains using angiotensin converting enzyme 2 (ACE2) as receptor are found. 

### summary zhang
[in work]
**Pangolin homology associated with 2019-nCoV**, Tao Zhang, Zhigang Zhang et al - Yunnan, 2020
#### Extracts
- "The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13"
- "We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively."


### summary k xiao
**Isolation of SARS-CoV-2-related coronavirus from Malayan pangolins**; Group: Wu Chen, Lihua Xiao, Yongyi Shen; Guangzhou
#### Methods & Results
Pangolins confiscated from two transports were investigated:
- The first transport in March 2019 contained 21 Malayan pangolins. During the following stay in a wildlife rescue center many gradually developed signs of a respiratory disease: shortness of breath, emaciation, lack of appetite, inactivity and crying. 17 of the 21 pangolins, were PCR positiv for a Sars-like coronavirus named Pangolin-CoV. 14 of the 17 infected pangolins died. Serology with an assay designed for S1 and RBD antigens of Sars-CoV-2 was done for 4 infected pangolins: Only one of the 4 sera was positive. 
- The second transport consisted of 4 Chinese and 4 Malayan pangolins. None of them was PCR positive for Sars-like coronaviruses.
#### Discussion
* Description of the genetic relations of Pangolin-CoV: Table 1 and Fig. 2. The genetic distances depend on the location on the genome and wheter nucleic or amino acids are considered:
    - Over the entire genome Pangolin-CoV has a sequence identity of 90.1% to Human-Sars-CoV-2, 88.9% to Bat-CoV RaTG13 and 88.0% to Bat-SARSr-CoV ZC45. 
    - The RBD of the spike gene is closest to Human-Sars-CoV-2 while at the 5' end of the S1 part of the spike the similarity is highest to at-SARSr-CoV ZC45.
* The varying similarities across the regions of the Pangolin-CoV genome indicate recombination events.   

### summary t lam 
#### Methods
* Investigation of frozen tissue samples (lungs, intestine and blood) from 18 Malayan pangolins (Manis javanica) obtained from anti-smuggling operations in Guangxi from August 2017 to January 2018. High-throughput RNA sequencing revealed the presence of coronaviruses in 6 out of 43 samples (2 lung samples, 2 intestinal samples, 1 lung–intes- tine mixed sample and 1 blood sample from 5 individual pangolins). Using amplicon sequencing six genome sequences are obtained. 
#### Results
* Six genome sequencess are obtained and named GX/P1E, GX/P2V, GX/P3B, GX/P4L, GX/P5E and GX/P5L.
#### Discussion
Comparison of the genomes obtained to SARS-like (Fig 1) 
and recombination analysis (Fig 2)             
    - a sliding window approach to identify breakpoints
    - Phylogenetic trees of the breakpoint free genomic regions
#### Extracts
* "two related lineages of coronaviruses are found in pangolins that were independently sampled in different Chinese provinces and that both are also related to SARS-CoV-2. This suggests that these animals may be important hosts for these viruses, which is surprising as pangolins are solitary animals that have relatively small population sizes, reflecting their endangered status"


### summary boni,lemey
**Evolutionary origins of the SARS‐CoV‐2 sarbecovirus lineage responsible for the COVID-19 pandemic**
Maciej F Boni , Philippe Lemey, Xiaowei Jiang , Tommy Tsan-Yuk Lam , Blair Perry , Todd Castoe , Andrew Rambaut and David L Robertson
#### Methods
Frequent recombination events makes genome analysis challenging. Regions likely without recombinations are identified using 3 different methods. The identified non-recombining regions are analysed using traditional phylogenetic methods including tree construction and dating. 
#### Results
* Aligning different sarbecovirus genomes, mainly sampled from bats, indicates frequent recombination events. 
* Phylogentic analysis on non-recombinant regions
    - This approach is used to estimate the dates of separation of different strains including Sars-CoV-1, Sars-CoV-2 and several strains found in bats.
    - Removing recombinant genomic segments point to a single ancestral lineage for SARS-CoV-2 and RaTG13 including Sars-like from pangolin (confiscated in March 2019 in Guangdong) and a clade of pangolin sequences from Guangxi province confiscated in 2017.
*  Evidence Sars-CoV-2 emerge directly from yet unknown reservoir (authors suggest bats):
    - SARS-CoV-2 and Pangolin-2019 are about 10% divergent throughout the entire S protein. 
    - RaTG13 and Sars-CoV-2 are similar in the S gene except for the variable loop region. RaTG13 is divergent in the variable loop region from Sars-CoV-2 and is the likely product of recombination, acquiring a divergent variable loop from an as yet unsampled bat sarbecovirus.



## Species Tropism

### summary zhai
**Comparison of Severe Acute Respiratory Syndrome Coronavirus 2 Spike Protein Binding to ACE2 Receptors from Human, Pets, Farm Animals, and Putative Intermediate Hosts**
#### Methods
* The binding of SARS-CoV-2 spikes to ACE 2 receptors of different animals is predicted based on X-ray structures of human ACE2 bound to the receptor-binding domain (RBD) of the spike protein (S)
* Animals include pets and farm animals
#### Results
* based on the number of key residue changes from human ACE 2, good binding is predicted for cat-like, sheep and cattle. 
* "In summary, binding of S of SARS-CoV-2 to ACE2 proteins tolerates a surprisingly large number of amino acid changes in the interaction surface, five in pig ACE2 and seven in civet ACE2."

### summary wu
**In Silico Analysis of Intermediate Hosts and Susceptible Animals of SARS-CoV-2**
#### Methods
Selection of 82 representative ACE2 sequences from the 1000 sequences with the closest homology to the human ACE2 protein. The selected ACE2 proteins were modeled by homology modeling and the binding free energy of ACE2 protein with the RBD of SARS-CoV-2 calculated <!--todo:lookup which method is used-->. 
#### Results
The ACE 2 receptors of primates and cat-like ACE are predicted to bind well to the SARS-2-ACE2-RBD.


## Coronaviruses and Insects

### Summary Tengs,Jonassen
**A mobile genetic element in the SARS-CoV-2 genome is shared wit multiple insect species**
#### Review of s2m knowledge
* s2m is a 41-43 nucleotide element always occuring at the 3’ of genomes
* the s2m hairpin-forming element
* s2m occurrence in viruses in patchy but if occurring the element is highly conserved.
* s2m is the only known example of a genetic element to move between distantly related viruses
* s2m is known to occur in Sarbecoviruses 
* s2m is a well conserved both in sequence and secondary structure
* unknown function
#### Methods
* all known s2m genotypes are searched using BLASTN in:
  * the entire virus section of GenBank to find out the presence in viruses
  * TSA and whole genome shotgun contigs databases to find out the presence in insects
* phylogenetic analysis using
  * ClustalW for aligning
  * MEGA X for maxiumum likelihood
  * James-Taylor-Thornton (JTT) substitution model
#### Results 
* 5553 s2m accessions
  * 3984 in SARS-Cov-2 strains
  * two similar sequences in virome of the spider species Tetragnatha maxillosa
  * winder moth and bagworm moth
* G>U mutation in position 31 which is only observed in the pandemic SARS-CoV-2 strains and in no other accessions s2m sequences
#### Discussion
* 10 base deletion inside s2m observed during passaging in Vero cell (experiments carried out by a different research group)



### Summary Watson 2002
**Mechanical Transmission of Turkey Coronavirus by Domestic Houseflies (Musca domestica Linnaeaus)**
Dawn Calibeo-Hayes, ..., D Wes Watson

#### Methods
* Turkey Coronaviruses (TCV) propagated in embryonated turkey eggs was used to prepare TCV inoculum.
* The groups flies were pooled. TCV in determined by inoculation of embroyonated turkey eggs and detection of TCV antigens 3 days later using immunofluorescent antibody technique
* Houseflies from a colony kept at North Carolina State University, 5 days flies used for the experiments. Flies were fed either with TCV innoculum (TCV exposed) or DMEM (sham exposed). 
* White Turkeys. Reared to an age of 7 days

#### Experiment 1
* Goal: Survival of TCV in fly crops
* Method: At 0.5, 1, 3, 6, 9 and 12 hours after the flies had consumed the innoculum the fly crops were investigated.
* Result: all fly crops contained infectious TCV after 6 hours after feeding. Some after 9 hours.
#### Experiment 2
* Goal: Survival of TCV in fly intestines 
* Method: as Experiment 1 but intestines were investigaed
* Result: no infectious TCV were present at any time in the intestines. Suggesting TCV were destroyed. 
#### Experiment 3
* Goal: Check if turkeys can get infected from eating TCV exposed houseflies
* Method: 0 (control),1,10 respectively 100 houseflies were fed to turkeys 3 hours after the flies consumed TCV inoculum. Infection of turkeys was determined directly by the presence of TCV antigens and indirectly by determine seroconversion. 
* Result: Some of the turkeys fed with only 1 fly were positive in antigen and seroconversion. Most of the turkeys fed with 10 or 100 flies were positive for both tests.